<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5.4 Project: Finding AMRs | Course Name</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="5.4 Project: Finding AMRs | Course Name" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5.4 Project: Finding AMRs | Course Name" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  



<meta name="date" content="2025-06-16" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/dasl_favicon.ico" type="image/x-icon" />
<link rel="prev" href="discussion-finding-amrs.html"/>
<link rel="next" href="presentation-finding-amrs.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="assets/style.css" type="text/css" />
<link rel="stylesheet" href="assets/C-MOOR_Template/c-moor-ottr.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.c-moor.org/" target="_blank"><img src="assets/C-MOOR_Template/c-moor-logo-vertical.png" style="width: 70%; padding-left: 10px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#learning-goals"><i class="fa fa-check"></i>Learning Goals</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="welcome-to-your-genomics-adventure.html"><a href="welcome-to-your-genomics-adventure.html"><i class="fa fa-check"></i><b>1.1</b> Welcome to Your Genomics Adventure!</a></li>
<li class="chapter" data-level="1.2" data-path="activity-create-accounts.html"><a href="activity-create-accounts.html"><i class="fa fa-check"></i><b>1.2</b> Activity: Create Accounts</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="activity-create-accounts.html"><a href="activity-create-accounts.html#purpose"><i class="fa fa-check"></i><b>1.2.1</b> Purpose</a></li>
<li class="chapter" data-level="1.2.2" data-path="activity-create-accounts.html"><a href="activity-create-accounts.html#activity"><i class="fa fa-check"></i><b>1.2.2</b> Activity</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="the-scientic-process.html"><a href="the-scientic-process.html"><i class="fa fa-check"></i><b>1.3</b> The Scientic Process</a></li>
<li class="chapter" data-level="1.4" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html"><i class="fa fa-check"></i><b>1.4</b> Homework: Post Introductions</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html#purpose-1"><i class="fa fa-check"></i><b>1.4.1</b> Purpose</a></li>
<li class="chapter" data-level="1.4.2" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html#learning-objectives"><i class="fa fa-check"></i><b>1.4.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="1.4.3" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html#introduction-1"><i class="fa fa-check"></i><b>1.4.3</b> Introduction</a></li>
<li class="chapter" data-level="1.4.4" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html#activity-1---create-a-topic-in-a-private-category"><i class="fa fa-check"></i><b>1.4.4</b> Activity 1 - Create a Topic in a Private Category</a></li>
<li class="chapter" data-level="1.4.5" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html#activity-2---reply-to-a-topic-in-a-public-category"><i class="fa fa-check"></i><b>1.4.5</b> Activity 2 - Reply to a Topic in a Public Category</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Exploration and Discovery</b></span></li>
<li class="chapter" data-level="2" data-path="scientific-literature.html"><a href="scientific-literature.html"><i class="fa fa-check"></i><b>2</b> Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.1" data-path="lecture-whats-in-your-xyz.html"><a href="lecture-whats-in-your-xyz.html"><i class="fa fa-check"></i><b>2.1</b> Lecture: What’s in Your XYZ?</a></li>
<li class="chapter" data-level="2.2" data-path="pre-lab-scientific-literature.html"><a href="pre-lab-scientific-literature.html"><i class="fa fa-check"></i><b>2.2</b> Pre-lab: Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="pre-lab-scientific-literature.html"><a href="pre-lab-scientific-literature.html#purpose-2"><i class="fa fa-check"></i><b>2.2.1</b> Purpose</a></li>
<li class="chapter" data-level="2.2.2" data-path="pre-lab-scientific-literature.html"><a href="pre-lab-scientific-literature.html#learning-objectives-1"><i class="fa fa-check"></i><b>2.2.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.2.3" data-path="pre-lab-scientific-literature.html"><a href="pre-lab-scientific-literature.html#introduction-2"><i class="fa fa-check"></i><b>2.2.3</b> Introduction</a></li>
<li class="chapter" data-level="2.2.4" data-path="pre-lab-scientific-literature.html"><a href="pre-lab-scientific-literature.html#activity-1"><i class="fa fa-check"></i><b>2.2.4</b> Activity</a></li>
<li class="chapter" data-level="2.2.5" data-path="pre-lab-scientific-literature.html"><a href="pre-lab-scientific-literature.html#footnotes-2"><i class="fa fa-check"></i><b>2.2.5</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="discussion-scientific-literature.html"><a href="discussion-scientific-literature.html"><i class="fa fa-check"></i><b>2.3</b> Discussion: Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="discussion-scientific-literature.html"><a href="discussion-scientific-literature.html#activity-2"><i class="fa fa-check"></i><b>2.3.1</b> Activity</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="lecture-scientific-literature.html"><a href="lecture-scientific-literature.html"><i class="fa fa-check"></i><b>2.4</b> Lecture: Scientific Literature</a></li>
<li class="chapter" data-level="2.5" data-path="activity-scientific-literature.html"><a href="activity-scientific-literature.html"><i class="fa fa-check"></i><b>2.5</b> Activity: Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="activity-scientific-literature.html"><a href="activity-scientific-literature.html#purpose-3"><i class="fa fa-check"></i><b>2.5.1</b> Purpose</a></li>
<li class="chapter" data-level="2.5.2" data-path="activity-scientific-literature.html"><a href="activity-scientific-literature.html#learning-objectives-2"><i class="fa fa-check"></i><b>2.5.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.5.3" data-path="activity-scientific-literature.html"><a href="activity-scientific-literature.html#introduction-3"><i class="fa fa-check"></i><b>2.5.3</b> Introduction</a></li>
<li class="chapter" data-level="2.5.4" data-path="activity-scientific-literature.html"><a href="activity-scientific-literature.html#activity-3"><i class="fa fa-check"></i><b>2.5.4</b> Activity</a></li>
<li class="chapter" data-level="2.5.5" data-path="activity-scientific-literature.html"><a href="activity-scientific-literature.html#results-in-class-and-homework"><i class="fa fa-check"></i><b>2.5.5</b> Results (in class and Homework)</a></li>
<li class="chapter" data-level="2.5.6" data-path="activity-scientific-literature.html"><a href="activity-scientific-literature.html#grading-criteria-3"><i class="fa fa-check"></i><b>2.5.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="2.5.7" data-path="activity-scientific-literature.html"><a href="activity-scientific-literature.html#footnotes-3"><i class="fa fa-check"></i><b>2.5.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="presentation-scientific-literature.html"><a href="presentation-scientific-literature.html"><i class="fa fa-check"></i><b>2.6</b> Presentation: Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="presentation-scientific-literature.html"><a href="presentation-scientific-literature.html#activity-4"><i class="fa fa-check"></i><b>2.6.1</b> Activity</a></li>
<li class="chapter" data-level="2.6.2" data-path="presentation-scientific-literature.html"><a href="presentation-scientific-literature.html#footnotes-4"><i class="fa fa-check"></i><b>2.6.2</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="microbial-genomes.html"><a href="microbial-genomes.html"><i class="fa fa-check"></i><b>3</b> Microbial Genomes</a>
<ul>
<li class="chapter" data-level="3.1" data-path="lecture-microbial-genome.html"><a href="lecture-microbial-genome.html"><i class="fa fa-check"></i><b>3.1</b> Lecture: Microbial Genome</a></li>
<li class="chapter" data-level="3.2" data-path="pre-lab-microbial-genomes.html"><a href="pre-lab-microbial-genomes.html"><i class="fa fa-check"></i><b>3.2</b> Pre-lab: Microbial Genomes</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="pre-lab-microbial-genomes.html"><a href="pre-lab-microbial-genomes.html#purpose-4"><i class="fa fa-check"></i><b>3.2.1</b> Purpose</a></li>
<li class="chapter" data-level="3.2.2" data-path="pre-lab-microbial-genomes.html"><a href="pre-lab-microbial-genomes.html#learning-objectives-3"><i class="fa fa-check"></i><b>3.2.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.2.3" data-path="pre-lab-microbial-genomes.html"><a href="pre-lab-microbial-genomes.html#activity-1-genbank"><i class="fa fa-check"></i><b>3.2.3</b> Activity 1 – GenBank</a></li>
<li class="chapter" data-level="3.2.4" data-path="pre-lab-microbial-genomes.html"><a href="pre-lab-microbial-genomes.html#activity-2-genomes-genes-and-other-databases-bv-brc"><i class="fa fa-check"></i><b>3.2.4</b> Activity 2 – Genomes, genes, and other databases (BV-BRC)</a></li>
<li class="chapter" data-level="3.2.5" data-path="pre-lab-microbial-genomes.html"><a href="pre-lab-microbial-genomes.html#activity-3---taxonomy-and-tree-of-life"><i class="fa fa-check"></i><b>3.2.5</b> Activity 3 - Taxonomy and tree of life</a></li>
<li class="chapter" data-level="3.2.6" data-path="pre-lab-microbial-genomes.html"><a href="pre-lab-microbial-genomes.html#grading-criteria-4"><i class="fa fa-check"></i><b>3.2.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="3.2.7" data-path="pre-lab-microbial-genomes.html"><a href="pre-lab-microbial-genomes.html#footnotes-5"><i class="fa fa-check"></i><b>3.2.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="discussion-microbial-genomes.html"><a href="discussion-microbial-genomes.html"><i class="fa fa-check"></i><b>3.3</b> Discussion: Microbial Genomes</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="discussion-microbial-genomes.html"><a href="discussion-microbial-genomes.html#activity-5"><i class="fa fa-check"></i><b>3.3.1</b> Activity</a></li>
<li class="chapter" data-level="3.3.2" data-path="discussion-microbial-genomes.html"><a href="discussion-microbial-genomes.html#footnotes-6"><i class="fa fa-check"></i><b>3.3.2</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="project-microbial-genomes.html"><a href="project-microbial-genomes.html"><i class="fa fa-check"></i><b>3.4</b> Project: Microbial Genomes</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="project-microbial-genomes.html"><a href="project-microbial-genomes.html#purpose-5"><i class="fa fa-check"></i><b>3.4.1</b> Purpose</a></li>
<li class="chapter" data-level="3.4.2" data-path="project-microbial-genomes.html"><a href="project-microbial-genomes.html#learning-objectives-4"><i class="fa fa-check"></i><b>3.4.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.4.3" data-path="project-microbial-genomes.html"><a href="project-microbial-genomes.html#activity-1-mags-and-taxonomy"><i class="fa fa-check"></i><b>3.4.3</b> Activity 1 – MAGs and Taxonomy</a></li>
<li class="chapter" data-level="3.4.4" data-path="project-microbial-genomes.html"><a href="project-microbial-genomes.html#activity-2-genomes-genes-and-databases"><i class="fa fa-check"></i><b>3.4.4</b> Activity 2 – Genomes, Genes, and Databases</a></li>
<li class="chapter" data-level="3.4.5" data-path="project-microbial-genomes.html"><a href="project-microbial-genomes.html#activity-3-go-deeper"><i class="fa fa-check"></i><b>3.4.5</b> Activity 3 – Go Deeper!</a></li>
<li class="chapter" data-level="3.4.6" data-path="project-microbial-genomes.html"><a href="project-microbial-genomes.html#grading-criteria-5"><i class="fa fa-check"></i><b>3.4.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="3.4.7" data-path="project-microbial-genomes.html"><a href="project-microbial-genomes.html#footnotes-7"><i class="fa fa-check"></i><b>3.4.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="presentation-microbial-genomes.html"><a href="presentation-microbial-genomes.html"><i class="fa fa-check"></i><b>3.5</b> Presentation: Microbial Genomes</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="presentation-microbial-genomes.html"><a href="presentation-microbial-genomes.html#activity-6"><i class="fa fa-check"></i><b>3.5.1</b> Activity</a></li>
<li class="chapter" data-level="3.5.2" data-path="presentation-microbial-genomes.html"><a href="presentation-microbial-genomes.html#footnotes-8"><i class="fa fa-check"></i><b>3.5.2</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Testing Ideas</b></span></li>
<li class="chapter" data-level="4" data-path="taxonomy-profiling.html"><a href="taxonomy-profiling.html"><i class="fa fa-check"></i><b>4</b> Taxonomy Profiling</a>
<ul>
<li class="chapter" data-level="4.1" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html"><i class="fa fa-check"></i><b>4.1</b> Activity: Taxonomy Profiling Spreadsheet</a>
<ul>
<li class="chapter" data-level="4.1.1" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#purpose-6"><i class="fa fa-check"></i><b>4.1.1</b> Purpose</a></li>
<li class="chapter" data-level="4.1.2" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#learning-objectives-5"><i class="fa fa-check"></i><b>4.1.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="4.1.3" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#introduction-5"><i class="fa fa-check"></i><b>4.1.3</b> Introduction</a></li>
<li class="chapter" data-level="4.1.4" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#activity-1-explore-zymo-gut-standard-metagenomic-diversity"><i class="fa fa-check"></i><b>4.1.4</b> Activity 1 – Explore Zymo Gut Standard Metagenomic Diversity</a></li>
<li class="chapter" data-level="4.1.5" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#activity-2-optional-compare-with-zymo-fecal-reference"><i class="fa fa-check"></i><b>4.1.5</b> Activity 2 (OPTIONAL) – Compare with Zymo Fecal Reference</a></li>
<li class="chapter" data-level="4.1.6" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#grading-criteria-6"><i class="fa fa-check"></i><b>4.1.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="4.1.7" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#footnotes-9"><i class="fa fa-check"></i><b>4.1.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="4.2" data-path="lecture-taxonomy-profiling.html"><a href="lecture-taxonomy-profiling.html"><i class="fa fa-check"></i><b>4.2</b> Lecture: Taxonomy Profiling</a></li>
<li class="chapter" data-level="4.3" data-path="prelab-taxonomy-profiling.html"><a href="prelab-taxonomy-profiling.html"><i class="fa fa-check"></i><b>4.3</b> Prelab: Taxonomy Profiling</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="prelab-taxonomy-profiling.html"><a href="prelab-taxonomy-profiling.html#purpose-7"><i class="fa fa-check"></i><b>4.3.1</b> Purpose</a></li>
<li class="chapter" data-level="4.3.2" data-path="prelab-taxonomy-profiling.html"><a href="prelab-taxonomy-profiling.html#learning-objectives-6"><i class="fa fa-check"></i><b>4.3.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="4.3.3" data-path="prelab-taxonomy-profiling.html"><a href="prelab-taxonomy-profiling.html#introduction-6"><i class="fa fa-check"></i><b>4.3.3</b> Introduction</a></li>
<li class="chapter" data-level="4.3.4" data-path="prelab-taxonomy-profiling.html"><a href="prelab-taxonomy-profiling.html#activity-1-qc-reads"><i class="fa fa-check"></i><b>4.3.4</b> Activity 1 – QC Reads</a></li>
<li class="chapter" data-level="4.3.5" data-path="prelab-taxonomy-profiling.html"><a href="prelab-taxonomy-profiling.html#activity-2-taxonomy-profiling-in-galaxy"><i class="fa fa-check"></i><b>4.3.5</b> Activity 2 – Taxonomy Profiling in Galaxy</a></li>
<li class="chapter" data-level="4.3.6" data-path="prelab-taxonomy-profiling.html"><a href="prelab-taxonomy-profiling.html#grading-criteria-7"><i class="fa fa-check"></i><b>4.3.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="4.3.7" data-path="prelab-taxonomy-profiling.html"><a href="prelab-taxonomy-profiling.html#footnotes-10"><i class="fa fa-check"></i><b>4.3.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="discussion-taxonomy-profiling.html"><a href="discussion-taxonomy-profiling.html"><i class="fa fa-check"></i><b>4.4</b> Discussion: Taxonomy Profiling</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="discussion-taxonomy-profiling.html"><a href="discussion-taxonomy-profiling.html#activity-7"><i class="fa fa-check"></i><b>4.4.1</b> Activity</a></li>
<li class="chapter" data-level="4.4.2" data-path="discussion-taxonomy-profiling.html"><a href="discussion-taxonomy-profiling.html#footnotes-11"><i class="fa fa-check"></i><b>4.4.2</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="project-taxonomy-profiling.html"><a href="project-taxonomy-profiling.html"><i class="fa fa-check"></i><b>4.5</b> Project: Taxonomy Profiling</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="project-taxonomy-profiling.html"><a href="project-taxonomy-profiling.html#purpose-8"><i class="fa fa-check"></i><b>4.5.1</b> Purpose</a></li>
<li class="chapter" data-level="4.5.2" data-path="project-taxonomy-profiling.html"><a href="project-taxonomy-profiling.html#learning-objectives-7"><i class="fa fa-check"></i><b>4.5.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="4.5.3" data-path="project-taxonomy-profiling.html"><a href="project-taxonomy-profiling.html#introduction-7"><i class="fa fa-check"></i><b>4.5.3</b> Introduction</a></li>
<li class="chapter" data-level="4.5.4" data-path="project-taxonomy-profiling.html"><a href="project-taxonomy-profiling.html#activity-1-qc"><i class="fa fa-check"></i><b>4.5.4</b> Activity 1 – QC</a></li>
<li class="chapter" data-level="4.5.5" data-path="project-taxonomy-profiling.html"><a href="project-taxonomy-profiling.html#activity-2-taxonomy-profiling"><i class="fa fa-check"></i><b>4.5.5</b> Activity 2 – Taxonomy Profiling</a></li>
<li class="chapter" data-level="4.5.6" data-path="project-taxonomy-profiling.html"><a href="project-taxonomy-profiling.html#grading-criteria-8"><i class="fa fa-check"></i><b>4.5.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="4.5.7" data-path="project-taxonomy-profiling.html"><a href="project-taxonomy-profiling.html#footnotes-12"><i class="fa fa-check"></i><b>4.5.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="presentation-taxonomy-profiling.html"><a href="presentation-taxonomy-profiling.html"><i class="fa fa-check"></i><b>4.6</b> Presentation: Taxonomy Profiling</a>
<ul>
<li class="chapter" data-level="4.6.1" data-path="presentation-taxonomy-profiling.html"><a href="presentation-taxonomy-profiling.html#activity-8"><i class="fa fa-check"></i><b>4.6.1</b> Activity</a></li>
<li class="chapter" data-level="4.6.2" data-path="presentation-taxonomy-profiling.html"><a href="presentation-taxonomy-profiling.html#instructions-33"><i class="fa fa-check"></i><b>4.6.2</b> Instructions</a></li>
<li class="chapter" data-level="4.6.3" data-path="presentation-taxonomy-profiling.html"><a href="presentation-taxonomy-profiling.html#footnotes-13"><i class="fa fa-check"></i><b>4.6.3</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="finding-amrs.html"><a href="finding-amrs.html"><i class="fa fa-check"></i><b>5</b> Finding AMRs</a>
<ul>
<li class="chapter" data-level="5.1" data-path="lecture-finding-amrs.html"><a href="lecture-finding-amrs.html"><i class="fa fa-check"></i><b>5.1</b> Lecture: Finding AMRs</a></li>
<li class="chapter" data-level="5.2" data-path="pre-lab-finding-amrs.html"><a href="pre-lab-finding-amrs.html"><i class="fa fa-check"></i><b>5.2</b> Pre-lab: Finding AMRs</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="pre-lab-finding-amrs.html"><a href="pre-lab-finding-amrs.html#purpose-9"><i class="fa fa-check"></i><b>5.2.1</b> Purpose</a></li>
<li class="chapter" data-level="5.2.2" data-path="pre-lab-finding-amrs.html"><a href="pre-lab-finding-amrs.html#learning-objectives-8"><i class="fa fa-check"></i><b>5.2.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.2.3" data-path="pre-lab-finding-amrs.html"><a href="pre-lab-finding-amrs.html#introduction-8"><i class="fa fa-check"></i><b>5.2.3</b> Introduction</a></li>
<li class="chapter" data-level="5.2.4" data-path="pre-lab-finding-amrs.html"><a href="pre-lab-finding-amrs.html#activity-1-flye-assembly"><i class="fa fa-check"></i><b>5.2.4</b> Activity 1 – Flye assembly</a></li>
<li class="chapter" data-level="5.2.5" data-path="pre-lab-finding-amrs.html"><a href="pre-lab-finding-amrs.html#activity-2-visualizing-contigs"><i class="fa fa-check"></i><b>5.2.5</b> Activity 2 – Visualizing contigs</a></li>
<li class="chapter" data-level="5.2.6" data-path="pre-lab-finding-amrs.html"><a href="pre-lab-finding-amrs.html#activity-3-finding-amrs"><i class="fa fa-check"></i><b>5.2.6</b> Activity 3 – Finding AMRs</a></li>
<li class="chapter" data-level="5.2.7" data-path="pre-lab-finding-amrs.html"><a href="pre-lab-finding-amrs.html#grading-criteria-9"><i class="fa fa-check"></i><b>5.2.7</b> Grading Criteria</a></li>
<li class="chapter" data-level="5.2.8" data-path="pre-lab-finding-amrs.html"><a href="pre-lab-finding-amrs.html#footnotes-14"><i class="fa fa-check"></i><b>5.2.8</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="5.3" data-path="discussion-finding-amrs.html"><a href="discussion-finding-amrs.html"><i class="fa fa-check"></i><b>5.3</b> Discussion: Finding AMRs</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="discussion-finding-amrs.html"><a href="discussion-finding-amrs.html#activity-9"><i class="fa fa-check"></i><b>5.3.1</b> Activity</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html"><i class="fa fa-check"></i><b>5.4</b> Project: Finding AMRs</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html#purpose-10"><i class="fa fa-check"></i><b>5.4.1</b> Purpose</a></li>
<li class="chapter" data-level="5.4.2" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html#learning-objectives-9"><i class="fa fa-check"></i><b>5.4.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="5.4.3" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html#introduction-9"><i class="fa fa-check"></i><b>5.4.3</b> Introduction</a></li>
<li class="chapter" data-level="5.4.4" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html#activity-1-genome-assembly-with-flye"><i class="fa fa-check"></i><b>5.4.4</b> Activity 1 – Genome assembly with Flye</a></li>
<li class="chapter" data-level="5.4.5" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html#activity-2-contig-visualization-with-bandage"><i class="fa fa-check"></i><b>5.4.5</b> Activity 2 – Contig visualization with Bandage</a></li>
<li class="chapter" data-level="5.4.6" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html#activity-3-finding-amrs-1"><i class="fa fa-check"></i><b>5.4.6</b> Activity 3 – Finding AMRs</a></li>
<li class="chapter" data-level="5.4.7" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html#activity-4-bin-contigs-with-metabat2"><i class="fa fa-check"></i><b>5.4.7</b> Activity 4 – Bin contigs with MetaBAT2</a></li>
<li class="chapter" data-level="5.4.8" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html#activity-5-read-about-gtdbtk-tool-in-galaxy"><i class="fa fa-check"></i><b>5.4.8</b> Activity 5 – Read about GTDBtk tool in Galaxy</a></li>
<li class="chapter" data-level="5.4.9" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html#grading-criteria-10"><i class="fa fa-check"></i><b>5.4.9</b> Grading Criteria</a></li>
<li class="chapter" data-level="5.4.10" data-path="project-finding-amrs.html"><a href="project-finding-amrs.html#footnotes-15"><i class="fa fa-check"></i><b>5.4.10</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="5.5" data-path="presentation-finding-amrs.html"><a href="presentation-finding-amrs.html"><i class="fa fa-check"></i><b>5.5</b> Presentation: Finding AMRs</a>
<ul>
<li class="chapter" data-level="5.5.1" data-path="presentation-finding-amrs.html"><a href="presentation-finding-amrs.html#activity-10"><i class="fa fa-check"></i><b>5.5.1</b> Activity</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="kickstart-project-work.html"><a href="kickstart-project-work.html"><i class="fa fa-check"></i><b>6</b> Kickstart Project Work</a>
<ul>
<li class="chapter" data-level="6.1" data-path="lecture-possible-datasets.html"><a href="lecture-possible-datasets.html"><i class="fa fa-check"></i><b>6.1</b> Lecture: Possible Datasets</a></li>
<li class="chapter" data-level="6.2" data-path="activity-possible-datasets.html"><a href="activity-possible-datasets.html"><i class="fa fa-check"></i><b>6.2</b> Activity: Possible Datasets</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="activity-possible-datasets.html"><a href="activity-possible-datasets.html#activity-11"><i class="fa fa-check"></i><b>6.2.1</b> Activity</a></li>
<li class="chapter" data-level="6.2.2" data-path="activity-possible-datasets.html"><a href="activity-possible-datasets.html#grading-criteria-11"><i class="fa fa-check"></i><b>6.2.2</b> Grading Criteria</a></li>
<li class="chapter" data-level="6.2.3" data-path="activity-possible-datasets.html"><a href="activity-possible-datasets.html#footnotes-17"><i class="fa fa-check"></i><b>6.2.3</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="project-work-organizer.html"><a href="project-work-organizer.html"><i class="fa fa-check"></i><b>6.3</b> Project Work Organizer</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="project-work-organizer.html"><a href="project-work-organizer.html#purpose-kickstart-project-work"><i class="fa fa-check"></i><b>6.3.1</b> Purpose: Kickstart Project Work</a></li>
<li class="chapter" data-level="6.3.2" data-path="project-work-organizer.html"><a href="project-work-organizer.html#purpose-project-work-organizer"><i class="fa fa-check"></i><b>6.3.2</b> Purpose: Project Work Organizer</a></li>
<li class="chapter" data-level="6.3.3" data-path="project-work-organizer.html"><a href="project-work-organizer.html#organizer-in-class-round-table-data-and-troubleshooting"><i class="fa fa-check"></i><b>6.3.3</b> Organizer: In Class Round Table Data and Troubleshooting</a></li>
<li class="chapter" data-level="6.3.4" data-path="project-work-organizer.html"><a href="project-work-organizer.html#organizer-individual-group-information"><i class="fa fa-check"></i><b>6.3.4</b> Organizer: Individual Group Information</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>TUTORIALS</b></span></li>
<li class="chapter" data-level="7" data-path="galaxy-tutorials.html"><a href="galaxy-tutorials.html"><i class="fa fa-check"></i><b>7</b> Galaxy Tutorials</a>
<ul>
<li class="chapter" data-level="7.1" data-path="a-short-introduction-to-galaxy.html"><a href="a-short-introduction-to-galaxy.html"><i class="fa fa-check"></i><b>7.1</b> A short introduction to Galaxy</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="a-short-introduction-to-galaxy.html"><a href="a-short-introduction-to-galaxy.html#introduction-10"><i class="fa fa-check"></i><b>7.1.1</b> Introduction</a></li>
<li class="chapter" data-level="7.1.2" data-path="a-short-introduction-to-galaxy.html"><a href="a-short-introduction-to-galaxy.html#activity-12"><i class="fa fa-check"></i><b>7.1.2</b> Activity</a></li>
<li class="chapter" data-level="7.1.3" data-path="a-short-introduction-to-galaxy.html"><a href="a-short-introduction-to-galaxy.html#grading-criteria-12"><i class="fa fa-check"></i><b>7.1.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="7.1.4" data-path="a-short-introduction-to-galaxy.html"><a href="a-short-introduction-to-galaxy.html#footnotes-18"><i class="fa fa-check"></i><b>7.1.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="qc-and-galaxy-workflows.html"><a href="qc-and-galaxy-workflows.html"><i class="fa fa-check"></i><b>7.2</b> QC and Galaxy Workflows</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="qc-and-galaxy-workflows.html"><a href="qc-and-galaxy-workflows.html#introduction-11"><i class="fa fa-check"></i><b>7.2.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2.2" data-path="qc-and-galaxy-workflows.html"><a href="qc-and-galaxy-workflows.html#activity-1-quality-control-qc"><i class="fa fa-check"></i><b>7.2.2</b> Activity 1 – Quality Control (QC)</a></li>
<li class="chapter" data-level="7.2.3" data-path="qc-and-galaxy-workflows.html"><a href="qc-and-galaxy-workflows.html#activity-2-creating-editing-and-importing-galaxy-workflows"><i class="fa fa-check"></i><b>7.2.3</b> Activity 2 – Creating, Editing and Importing Galaxy Workflows</a></li>
<li class="chapter" data-level="7.2.4" data-path="qc-and-galaxy-workflows.html"><a href="qc-and-galaxy-workflows.html#grading-criteria-13"><i class="fa fa-check"></i><b>7.2.4</b> Grading Criteria</a></li>
<li class="chapter" data-level="7.2.5" data-path="qc-and-galaxy-workflows.html"><a href="qc-and-galaxy-workflows.html#footnotes-19"><i class="fa fa-check"></i><b>7.2.5</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="galaxy-data-collections.html"><a href="galaxy-data-collections.html"><i class="fa fa-check"></i><b>7.3</b> Galaxy data collections</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="galaxy-data-collections.html"><a href="galaxy-data-collections.html#introduction-12"><i class="fa fa-check"></i><b>7.3.1</b> Introduction</a></li>
<li class="chapter" data-level="7.3.2" data-path="galaxy-data-collections.html"><a href="galaxy-data-collections.html#activity-13"><i class="fa fa-check"></i><b>7.3.2</b> Activity</a></li>
<li class="chapter" data-level="7.3.3" data-path="galaxy-data-collections.html"><a href="galaxy-data-collections.html#grading-criteria-14"><i class="fa fa-check"></i><b>7.3.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="7.3.4" data-path="galaxy-data-collections.html"><a href="galaxy-data-collections.html#footnotes-20"><i class="fa fa-check"></i><b>7.3.4</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="8" data-path="sciserver-tutorials.html"><a href="sciserver-tutorials.html"><i class="fa fa-check"></i><b>8</b> SciServer Tutorials</a>
<ul>
<li class="chapter" data-level="8.1" data-path="meet-r.html"><a href="meet-r.html"><i class="fa fa-check"></i><b>8.1</b> Meet R!</a>
<ul>
<li class="chapter" data-level="8.1.1" data-path="meet-r.html"><a href="meet-r.html#introduction-13"><i class="fa fa-check"></i><b>8.1.1</b> Introduction</a></li>
<li class="chapter" data-level="8.1.2" data-path="meet-r.html"><a href="meet-r.html#activity-14"><i class="fa fa-check"></i><b>8.1.2</b> Activity</a></li>
<li class="chapter" data-level="8.1.3" data-path="meet-r.html"><a href="meet-r.html#grading-criteria-15"><i class="fa fa-check"></i><b>8.1.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="8.1.4" data-path="meet-r.html"><a href="meet-r.html#footnotes-21"><i class="fa fa-check"></i><b>8.1.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="8.2" data-path="r-for-data-science-1.html"><a href="r-for-data-science-1.html"><i class="fa fa-check"></i><b>8.2</b> R for Data Science 1</a>
<ul>
<li class="chapter" data-level="8.2.1" data-path="r-for-data-science-1.html"><a href="r-for-data-science-1.html#introduction-14"><i class="fa fa-check"></i><b>8.2.1</b> Introduction</a></li>
<li class="chapter" data-level="8.2.2" data-path="r-for-data-science-1.html"><a href="r-for-data-science-1.html#activity-15"><i class="fa fa-check"></i><b>8.2.2</b> Activity</a></li>
<li class="chapter" data-level="8.2.3" data-path="r-for-data-science-1.html"><a href="r-for-data-science-1.html#grading-criteria-16"><i class="fa fa-check"></i><b>8.2.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="8.2.4" data-path="r-for-data-science-1.html"><a href="r-for-data-science-1.html#footnotes-22"><i class="fa fa-check"></i><b>8.2.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="8.3" data-path="test-driver.html"><a href="test-driver.html"><i class="fa fa-check"></i><b>8.3</b> test-driveR</a>
<ul>
<li class="chapter" data-level="8.3.1" data-path="test-driver.html"><a href="test-driver.html#introduction-15"><i class="fa fa-check"></i><b>8.3.1</b> Introduction</a></li>
<li class="chapter" data-level="8.3.2" data-path="test-driver.html"><a href="test-driver.html#activity-16"><i class="fa fa-check"></i><b>8.3.2</b> Activity</a></li>
<li class="chapter" data-level="8.3.3" data-path="test-driver.html"><a href="test-driver.html#grading-criteria-17"><i class="fa fa-check"></i><b>8.3.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="8.3.4" data-path="test-driver.html"><a href="test-driver.html#footnotes-23"><i class="fa fa-check"></i><b>8.3.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="r-for-data-science-2.html"><a href="r-for-data-science-2.html"><i class="fa fa-check"></i><b>8.4</b> R for Data Science 2</a>
<ul>
<li class="chapter" data-level="8.4.1" data-path="r-for-data-science-2.html"><a href="r-for-data-science-2.html#introduction-16"><i class="fa fa-check"></i><b>8.4.1</b> Introduction</a></li>
<li class="chapter" data-level="8.4.2" data-path="r-for-data-science-2.html"><a href="r-for-data-science-2.html#activity-17"><i class="fa fa-check"></i><b>8.4.2</b> Activity</a></li>
<li class="chapter" data-level="8.4.3" data-path="r-for-data-science-2.html"><a href="r-for-data-science-2.html#grading-criteria-18"><i class="fa fa-check"></i><b>8.4.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="8.4.4" data-path="r-for-data-science-2.html"><a href="r-for-data-science-2.html#footnotes-24"><i class="fa fa-check"></i><b>8.4.4</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>COMMUNITY ANALYSIS AND FEEDBACK</b></span></li>
<li class="chapter" data-level="9" data-path="scientific-posters.html"><a href="scientific-posters.html"><i class="fa fa-check"></i><b>9</b> Scientific Posters</a>
<ul>
<li class="chapter" data-level="9.1" data-path="lecture-scientific-posters.html"><a href="lecture-scientific-posters.html"><i class="fa fa-check"></i><b>9.1</b> Lecture: Scientific Posters</a></li>
<li class="chapter" data-level="9.2" data-path="activity-scientific-posters.html"><a href="activity-scientific-posters.html"><i class="fa fa-check"></i><b>9.2</b> Activity: Scientific Posters</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="activity-scientific-posters.html"><a href="activity-scientific-posters.html#activity-18"><i class="fa fa-check"></i><b>9.2.1</b> Activity</a></li>
<li class="chapter" data-level="9.2.2" data-path="activity-scientific-posters.html"><a href="activity-scientific-posters.html#grading-criteria-19"><i class="fa fa-check"></i><b>9.2.2</b> Grading Criteria</a></li>
<li class="chapter" data-level="9.2.3" data-path="activity-scientific-posters.html"><a href="activity-scientific-posters.html#footnotes-25"><i class="fa fa-check"></i><b>9.2.3</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by:</a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Course Name</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=10.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/C-MOOR_Template/dasl_thin_main_image.png">
</div>
<div id="project-finding-amrs" class="section level2 hasAnchor" number="5.4">
<h2><span class="header-section-number">5.4</span> Project: Finding AMRs<a href="project-finding-amrs.html#project-finding-amrs" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="purpose-10" class="section level3 hasAnchor" number="5.4.1">
<h3><span class="header-section-number">5.4.1</span> Purpose<a href="project-finding-amrs.html#purpose-10" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>To explore soil metagenomics using Galaxy tools for de novo genome assembly (Flye), assembled contig visualization (Bandage), finding antimicrobial resistance (AMR) genes (ABRicate) and binning contigs into larger groups/bins (MetaBAT2). You will also explore information about the taxonomic classifier GTDB-Tk. This will allow you to compare genomic assemblies and AMRs between soil and Zymo gut standard.</p>
</div>
<div id="learning-objectives-9" class="section level3 hasAnchor" number="5.4.2">
<h3><span class="header-section-number">5.4.2</span> Learning Objectives<a href="project-finding-amrs.html#learning-objectives-9" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ol style="list-style-type: decimal">
<li>Use <strong>Galaxy</strong> tools to</li>
<li>Use <strong>Flye</strong> tool to perform de novo genome assembly of long reads into ‘contigs’</li>
<li>Visualize the contigs with <strong>Bandage</strong> visualization tool</li>
<li>Use <strong>ABRicate</strong> tool to find AMRs in contig assemblies</li>
<li>Use <strong>MetaBAT2</strong> to bin contigs into larger MAGs</li>
<li>Learn about <strong>GTDB-Tk</strong> taxonomy classifier tool</li>
</ol>
</div>
<div id="introduction-9" class="section level3 hasAnchor" number="5.4.3">
<h3><span class="header-section-number">5.4.3</span> Introduction<a href="project-finding-amrs.html#introduction-9" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>In this activity we will practice de novo genome assembly with Flye, using soil metagenomes from Nanopore sequencing . You will have an opportunity to compare and contrast assembled contigs and antimicrobial resistance profiles of soil and gut and think about the differences and similarities in microbial diversity of the two environments.</p>
<p><br></p>
<p>In this project you will also learn about MetaBAT2 and GTDB-Tk. Let’s start now. The most up to date long read metagenomics workflow includes contig assembly (e.g. using Flye tool), followed by contig binning into larger metagenome-assembled genomes (MAGs) (with e.g. MetaBAT2 tool) and finally MAG classification (e.g. with GTDB-Tk). MetaBAT2 is an algorithm that bins (or groups) sequence fragments (contigs) into larger MAGs or draft genomes. Subsequently, MAGs can be taxonomically classified by GTDB-Tk.</p>
<p><br></p>
<p><code>You will not be executing GTDB-Tk in this activity to stay within a reasonable activity time frame.</code> However, for your project work, you will have a chance to test GTDB-Tk on your genome assemblies and bins. A note to your future self working on a project - as a rule, after genome assembly, if your contigs are 500 kb or above, they will be considered large enough to be passed to GTDB-Tk without the need for binning. Contigs of &lt; 500 kb will be binned and passed on to GTDB-Tk for taxonomy classification as bins.</p>
<p><strong>Table of approximate minimum times for a job to be completed on Galaxy using specified tools.</strong></p>
<ul>
<li>Note, these times apply only to the specific input file we will be using in this activity, the nanopore-soil-subset that is 5.4 GB</li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">Flye</th>
<th align="left">Bandage</th>
<th align="left">ABRicate</th>
<th align="left">MetaBAT2</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">5 hours</td>
<td align="left">&lt; 5 min</td>
<td align="left">&lt; 10 min</td>
<td align="left">&lt; 10 min</td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="activity-1-genome-assembly-with-flye" class="section level3 hasAnchor" number="5.4.4">
<h3><span class="header-section-number">5.4.4</span> Activity 1 – Genome assembly with Flye<a href="project-finding-amrs.html#activity-1-genome-assembly-with-flye" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 30 min (activity time DOES NOT include the Flye run time on Galaxy)</em></p>
<div id="instructions-38" class="section level4 hasAnchor" number="5.4.4.1">
<h4><span class="header-section-number">5.4.4.1</span> Instructions<a href="project-finding-amrs.html#instructions-38" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ol style="list-style-type: decimal">
<li>Run Flye in Galaxy on quality filtered (with fastp tool) nanopore soil pilot subset <a href="https://usegalaxy.org/u/valerie-g/h/nanopore-soil-subset-filtered">nanopore-soil-subset-filtered</a> to de novo assemble soil microbial genomes.</li>
</ol>
<ol style="list-style-type: lower-alpha">
<li>Obtain .fastq file from a subset of Nanopore-sequencing soil study</li>
</ol>
<ul>
<li><a href="%5Bhttps://usegalaxy.org/u/valerie-g/h/nanopore-soil-subset-filtered">https://usegalaxy.org/u/valerie-g/h/nanopore-soil-subset-filtered</a></li>
</ul>
<ol start="2" style="list-style-type: lower-alpha">
<li><p>Name your new history <strong>“Finding soil AMRs”</strong></p></li>
<li><p>Run <strong>Flye</strong> tool to assess sequence quality using the following <strong>Tool Parameters</strong></p></li>
</ol>
<ul>
<li>Under <strong>Input Reads</strong>: select your nano <code>pore-soil-subset-filtered</code> <strong>.fastq</strong> dataset.</li>
<li>Under <strong>Mode</strong>: select <code>--nano-raw</code> option, since the sequences were obtained using Nanopore sequencing technology.</li>
<li>Under <strong>Perform metagenomic assembly</strong>: select <code>Yes</code></li>
<li>Under <strong>Generate a log file</strong>: select <code>Yes</code></li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Explore Flye output <strong>assembly info</strong> file which is sorted by length (in base pairs, bp) of the contig (high to low).</li>
</ol>
</div>
<div id="questions-26" class="section level4 hasAnchor" number="5.4.4.2">
<h4><span class="header-section-number">5.4.4.2</span> Questions<a href="project-finding-amrs.html#questions-26" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1. How many contigs were assembled?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"><em>Note: Since each contig is represented by a separate row (or line) in the assembly info file, simply clicking on the assembly info file and recording the number of lines listed in the file will correspond to the number of contigs</em></td>
</tr>
<tr class="even">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">2. What is the longest contig size?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3. What percent of input was assembled into contigs?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">- <em>Note 1: Based on the log file, the input going into flye assembly was 6,103,654,873 bases.</em></td>
</tr>
<tr class="even">
<td align="left">- <em>Note 2: Based on the log file, the output going of flye assembly was 154,251,885 bases.</em></td>
</tr>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">4. Why do you think only a small fraction of reads was assembled into contigs?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>5. Compare soil assembly to the Zymo gut standard assembly provided the following observations:</strong></p>
<ul>
<li>For this activity you can consult back to your Prelab: Finding AMRs.</li>
<li>The number of contigs assembled from <strong>339.9 MB</strong> of sequencing data from the Zymo gut standard D6331 subset was <strong>265</strong>, much smaller than for the filtered soil sample of 5.4 GB! Yet, the largest contig size for the Zymo gut standard was <strong>2,158,044</strong> (almost 9 times larger) and circular (while the largest contig for the soil sample was linear).</li>
</ul>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">5A. Why do you think the number of contigs in the soil sample was so much higher than the number of contigs in the Zymo gut standard?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">- <em>Hint 1 - it is NOT because of the difference in the size of the sequencing file.</em></td>
</tr>
<tr class="even">
<td align="left">- <em>Hint 2 - Think about possible differences in the microbial diversity of the two samples.</em></td>
</tr>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">5B. Why do you think it was possible to assemble a much larger and circular contig with the Zymo gut standard sample compared to the soil sample?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
</div>
<div id="activity-2-contig-visualization-with-bandage" class="section level3 hasAnchor" number="5.4.5">
<h3><span class="header-section-number">5.4.5</span> Activity 2 – Contig visualization with Bandage<a href="project-finding-amrs.html#activity-2-contig-visualization-with-bandage" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 15 min </em></p>
<div id="instructions-39" class="section level4 hasAnchor" number="5.4.5.1">
<h4><span class="header-section-number">5.4.5.1</span> Instructions<a href="project-finding-amrs.html#instructions-39" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ol style="list-style-type: decimal">
<li>Run <strong>Bandage</strong> Image tool in Galaxy to visualize contigs.</li>
</ol>
<ul>
<li>Run <strong>Bandage Image</strong> tool in Galaxy using your <strong>Flye: graphical fragment assembly file</strong> (in gfa1 format) as input, using default parameters.</li>
</ul>
</div>
<div id="questions-27" class="section level4 hasAnchor" number="5.4.5.2">
<h4><span class="header-section-number">5.4.5.2</span> Questions<a href="project-finding-amrs.html#questions-27" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<table>
<thead>
<tr class="header">
<th align="left">1. Paste the resulting image below.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">2. Describe contig profile based on the Bandage Image result.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
</div>
<div id="activity-3-finding-amrs-1" class="section level3 hasAnchor" number="5.4.6">
<h3><span class="header-section-number">5.4.6</span> Activity 3 – Finding AMRs<a href="project-finding-amrs.html#activity-3-finding-amrs-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 30 min </em></p>
<div id="instructions-40" class="section level4 hasAnchor" number="5.4.6.1">
<h4><span class="header-section-number">5.4.6.1</span> Instructions<a href="project-finding-amrs.html#instructions-40" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ol style="list-style-type: decimal">
<li>Run <strong>ABRicate</strong> tool in Galaxy using Flye consensus as input using the following <strong>Tool Parameters</strong>:</li>
</ol>
<ul>
<li>Under <strong>Input Reads</strong>: select your <code>Flye: consensus</code> output in FASTA format</li>
<li>IMPORTANT: Under <strong>Advanced Options</strong>: select <code>NCBI Bacterial Antimicrobial Resistance Reference Gene Database</code> as your database option; the default ‘resfinder’ may not work well.</li>
</ul>
<ol start="2" style="list-style-type: decimal">
<li>Explore <strong>ABRicate report</strong> file.</li>
</ol>
<ul>
<li>Note, Abricate output report has the following information:</li>
</ul>
<table>
<colgroup>
<col width="50%" />
<col width="50%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Column</th>
<th align="left">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">FILE</td>
<td align="left">The filename this hit came from</td>
</tr>
<tr class="even">
<td align="left">SEQUENCE</td>
<td align="left">The sequence in the filename</td>
</tr>
<tr class="odd">
<td align="left">START</td>
<td align="left">Start coordinate in the sequence</td>
</tr>
<tr class="even">
<td align="left">END</td>
<td align="left">End coordinate in the sequence</td>
</tr>
<tr class="odd">
<td align="left">GENE</td>
<td align="left">ABR gene name</td>
</tr>
<tr class="even">
<td align="left">COVERAGE</td>
<td align="left">What proportion of the gene is in our sequence</td>
</tr>
<tr class="odd">
<td align="left">COVERAGE_MAP</td>
<td align="left">A visual represenation of coverage map (gaps or no gaps)</td>
</tr>
<tr class="even">
<td align="left">GAPS</td>
<td align="left">Was there any gaps in the alignment - possible pseudogene?</td>
</tr>
<tr class="odd">
<td align="left">%COVERAGE</td>
<td align="left">Proportion of gene covered</td>
</tr>
<tr class="even">
<td align="left">%IDENTITY</td>
<td align="left">Proportion of exact nucleotide matches</td>
</tr>
<tr class="odd">
<td align="left">DATABASE</td>
<td align="left">The database this sequence comes from</td>
</tr>
<tr class="even">
<td align="left">ACCESSION</td>
<td align="left">The genomic source of the sequence</td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="questions-28" class="section level4 hasAnchor" number="5.4.6.2">
<h4><span class="header-section-number">5.4.6.2</span> Questions<a href="project-finding-amrs.html#questions-28" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1. How many AMR genes were detected? This is the number of rows in your file.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2. How many DIFFERENT AMR genes were detected and what are their GENE names?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3. What are the different AMR genes resistant to? What is their RESISTANCE?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">4. How many DIFFERENT contigs had AMRs?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>5. Research and write a small paragraph report on one of the AMR genes you found.</strong></p>
<ul>
<li>Use any search tools for your research, but we encourage you to use PubMed <a href="https://pubmed.ncbi.nlm.nih.gov/">https://pubmed.ncbi.nlm.nih.gov/</a> where you can find many scientific articles on the topic if you search for e.g. your AMR gene name, or resistance name or using a sentence as input.</li>
<li>Talk about anything of interest, e.g., which microbes have the AMR of interest, what is the substance to which the gene shows resistance to, where could the resistance to this substance come from, what are possible health implications, etc.</li>
</ul>
<table>
<thead>
<tr class="header">
<th align="left">5A. Report on one of the AMR genes you found.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">5B. Why do you think the AMRs you found in soil differ from the AMRs you found in the gut (from your pre-lab)?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
</div>
<div id="activity-4-bin-contigs-with-metabat2" class="section level3 hasAnchor" number="5.4.7">
<h3><span class="header-section-number">5.4.7</span> Activity 4 – Bin contigs with MetaBAT2<a href="project-finding-amrs.html#activity-4-bin-contigs-with-metabat2" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 20 min </em></p>
<div id="instructions-41" class="section level4 hasAnchor" number="5.4.7.1">
<h4><span class="header-section-number">5.4.7.1</span> Instructions<a href="project-finding-amrs.html#instructions-41" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ol style="list-style-type: decimal">
<li>In Galaxy, find and click on <strong>MetaBAT2</strong> tool and explore tool parameters.</li>
<li>Run <strong>MetaBAT2</strong> tool in Galaxy using <code>Flye consensus</code> as input using the <strong>following Tool Parameters</strong>:</li>
</ol>
<ul>
<li>Under <strong>Fasta file containing contigs</strong>: select your Flye: consensus output in FASTA format</li>
<li>Under <strong>Output options</strong>: from the <strong>Extra outputs</strong> dropdown menu select: <code>Process log file</code>.</li>
</ul>
</div>
<div id="questions-29" class="section level4 hasAnchor" number="5.4.7.2">
<h4><span class="header-section-number">5.4.7.2</span> Questions<a href="project-finding-amrs.html#questions-29" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><strong>1. Explore MetaBAT2 tool and parameters.</strong></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1A. What is the function of MetaBAT2 tool based on Galaxy tool description on top?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1B. Under Tool Parameters for MetaBAT2, find and record below the Minimum size of a contig for binning (a value given in basepairs, bp).</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1C. Under Tool Parameters and Output options for MetaBAT2, find and record below the Minimum size of a bin as the output.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>2. Explore MetaBAT2 tool output.</strong></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2A. Open MetaBAT2 Process log output file and record how many bins were formed from contigs.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2B. Open MetaBAT2 Process log output file and record how many bases in total were used to form bins.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2C. What percent of contig bases formed bins (given that 154,251,885 bases were in Flye output)?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2D. Based on percent of contigs that formed bins (from activity 4-2.1 above) did metaBAT2 do a good job of binning the contigs?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>3. Examine MetaBAT2 Bin sequences output, which is a <code>DATASET COLLECTION</code>, where each collection is a separate bin.</strong></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3A. Click on the MetaBAT2 Bin sequences output. How many bins (or Galaxy ‘folders’) are there?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3B. Click on the MetaBAT2 Bin sequences output and then on bin 1. Without ‘eyeballing’ the fasta file, note how many sequences (contigs) were included in bin 1.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3C. Click on the MetaBAT2 Bin sequences output and then on bin 2. Without ‘eyeballing’ the fasta file, note how many sequences (contigs) were included in bin 2?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
</div>
<div id="activity-5-read-about-gtdbtk-tool-in-galaxy" class="section level3 hasAnchor" number="5.4.8">
<h3><span class="header-section-number">5.4.8</span> Activity 5 – Read about GTDBtk tool in Galaxy<a href="project-finding-amrs.html#activity-5-read-about-gtdbtk-tool-in-galaxy" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 15 min </em></p>
<div id="instructions-42" class="section level4 hasAnchor" number="5.4.8.1">
<h4><span class="header-section-number">5.4.8.1</span> Instructions<a href="project-finding-amrs.html#instructions-42" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ol style="list-style-type: decimal">
<li>In Galaxy, find and click on <strong>GTDB-Tk Classify genomes.</strong></li>
</ol>
</div>
<div id="questions-30" class="section level4 hasAnchor" number="5.4.8.2">
<h4><span class="header-section-number">5.4.8.2</span> Questions<a href="project-finding-amrs.html#questions-30" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1. Read the GTDB-Tk Classify genomes tool’s “What it does” part and summarize what GTDB-Tk does.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2. What is the ideal input sequence for GTDB-Tk classification: 1) raw sequences, 2) contigs or 3) large contigs(&gt;500kb) and MAGs?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<p><strong>3. Visit <a href="https://gtdb.ecogenomic.org/stats/r220">https://gtdb.ecogenomic.org/stats/r220</a> to explore the database and database statistics.</strong></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3A. How many bacterial species are present in GTDB database Release 220?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3B. Scroll through website . Although it has a lot of complex information, what is one thing you found interesting about GTDB-Tk content?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
</div>
<div id="grading-criteria-10" class="section level3 hasAnchor" number="5.4.9">
<h3><span class="header-section-number">5.4.9</span> Grading Criteria<a href="project-finding-amrs.html#grading-criteria-10" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li><mark style="background-color: yellow">Download as Microsoft Word (.docx) and upload on Canvas</li>
</ul>
</div>
<div id="footnotes-15" class="section level3 hasAnchor" number="5.4.10">
<h3><span class="header-section-number">5.4.10</span> Footnotes<a href="project-finding-amrs.html#footnotes-15" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Resources</strong></p>
<ul>
<li>Google Doc</li>
</ul>
<p><strong>Contributions and Affiliations</strong></p>
<ul>
<li>Valeriya Gaysinskaya, Johns Hopkins University</li>
<li>Frederick Tan, Johns Hopkins University</li>
</ul>
<p>Last Revised: June 2025</p>
</div>
</div>
<hr>
<center> 
  <div class="footer">
      All materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="discussion-finding-amrs.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="presentation-finding-amrs.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
