```{r, include = FALSE}
ottrpal::set_knitr_image_path()
```
# (PART\*) Testing Ideas {-}

# Taxonomy Profiling

## Lecture: Taxonomy Profiling

```{r, echo=FALSE, fig.alt=''}
ottrpal::include_slide("https://docs.google.com/presentation/d/10P4ktKWSrRpM1YAWYW31tLly_jf1rcfgCMbzZ9Tx9xY/edit?slide=id.g35f391192_00#slide=id.g35f391192_00")
```

[Slides: Taxonomy Profiling](https://docs.google.com/presentation/d/10P4ktKWSrRpM1YAWYW31tLly_jf1rcfgCMbzZ9Tx9xY/edit?usp=sharing)


## Homework: Taxonomy Profiling Prelab

### Purpose

To use a variety of Galaxy tools to perform Quality Control (QC), taxonomy profiling, and visualization of a metagenomics sample.

### Learning Objectives

Use Galaxy tools to:

1. Perform Quality Control (QC) on your raw data by checking the quality of your raw reads
2. Assign taxonomy labels to your reads
3. Visualize the classified metagenome

In this exercise, using Galaxy, you will be:

1. Using NanoPlot tool to examine the quality of your sequencing reads
2. Running a workflow to perform taxonomy profiling and visualization in a single step

### Introduction

To find out which microorganisms are present in the sample, it is important to have high-quality DNA sequences. To ensure high-quality sequence input, QC (and in many cases also read trimming and filtering) are routinely performed on raw sequences. The reads can then be used to determine which species, genera, families, and other taxonomic ranks are present in the sample. To assign taxonomy, we can compare the reads of the sample to a reference database, i.e. sequences of known microorganisms stored in a database, using Kraken2, which is a k-mer based taxonomic assignment tool. We can then use a tool like Krona to interactively visualize and explore the composition of a metagenome.  

### Activity 1 – QC Reads

*Estimated time: 50 min*

#### Activity 1 - Part I: Import dataset into Galaxy

*Estimated time: 15 min*

#### Instructions

Import dataset into Galaxy. 

a. Open the zymo-gut-standard public history [https://usegalaxy.org/u/valerie-g/h/zymo-gut-standard-d6331-subset-1](https://usegalaxy.org/u/valerie-g/h/zymo-gut-standard-d6331-subset-1) 

b. Click on **Import this history**, select Copy only the active, non-deleted datasets and then Copy History. 

c. Confirm <mark style="background color: lightgreen">Zymo_Gut_Standard_D6331_subset exists in your history by clicking on the Home button "Galaxy" on top left ().

d. Click on Zymo_Gut_Standard_D6331_subset to explore content.


#### Questions

|1. What is the size of this downloaded dataset subset?|
|:--|
|<break>|

|2. What is the format/extension of the downloaded file?|
|:--|
|<break>|

|3. Click on the Display (eyeball) icon and describe what you see in lines 1, 2, 3 and 4 of the fastq file?| |
|:--|:--|
|Line 1:| |
|Line 2:| |
|Line 3:| |
|Line 4:| |
|<break>| |

#### Activity 1 - Part II: Run Nanoplot to assess sequence quality

*Estimated time: 15 min*

#### Instructions

In Galaxy, click on the Tools icon on the left of the page. Then, in the search bar enter ‘NanoPlot’ and select the **NanoPlot** tool. Explore NanoPlot functionality via examining **Tool Parameters**.

1. Under **Type of file(s) to work on**, check  to see which input files are compatible with NanoPlot and name 2 file extension options listed.

| **Record NanoPlot input file options**| |
|:--|:--|
|File extension name 1:| |
|File extension name 2:| |
|<br>| 

2. Click to expand *Options for filtering or transforming input prior to plotting* and name 3 options you could use to filter your sequencing data.

| **Record NanoPlot sequencing data filtering options**| |
|:--|:--|
|Option 1:| |
|Option 2:| |
|Option 3:| |
|<br>| 

3. <mark style="background color: yellow">**Run Nanoplot** using default tool settings. Under Tool Parameters, <mark style="background color: lightyellow">**check the following settings**:

- Under files * There are 3 options to select a fastq dataset: Single dataset, Multiple datasets, or Dataset collection; Browse to select your fastq dataset by clicking on the ‘...’ tab for a Single dataset option. 
- Note, Galaxy tool may pre-select the correct dataset already for you so just make sure that the file is correct. 
- Click on Run Tool and wait ~5-10 minutes as the NanoPlot job is scheduled, run, and complete.

Record how many output files you obtained and list their names.

| **Record how many output files you obtained after running NanoPlot and list their names.| 
|:--|:--|
| |
| |
| |
|<br>|  


#### Activity 1 - Part III: View NanoPlot Results

*Estimated time: 15 min*

#### Instructions




## Lecture: Possible Datasets

```{r, echo=FALSE, fig.alt=''}
ottrpal::include_slide("https://docs.google.com/presentation/d/1VxwSmAY8BUs3EfVcxPm3I8kNYJWjVqoHJrGOX3X3sog/edit?slide=id.g35f391192_00#slide=id.g35f391192_00")
```

[Slides: Possible Datasets](https://docs.google.com/presentation/d/1VxwSmAY8BUs3EfVcxPm3I8kNYJWjVqoHJrGOX3X3sog/edit?usp=sharing)


## Homework: Possible Datasets Activity


## Homework: Using dataset collections


## Discussion: Taxonomy Profiling Prelab


## Homework: Taxonomy Profiling Project

## Activity: Meet R!

## Presentation: Taxonomy Profiling Project

