<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>4.2 Homework: Taxonomy Profiling Prelab | Course Name</title>
  <meta name="description" content="Description about Course/Book." />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="4.2 Homework: Taxonomy Profiling Prelab | Course Name" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="Description about Course/Book." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="4.2 Homework: Taxonomy Profiling Prelab | Course Name" />
  
  <meta name="twitter:description" content="Description about Course/Book." />
  



<meta name="date" content="2025-05-21" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  <link rel="shortcut icon" href="assets/dasl_favicon.ico" type="image/x-icon" />
<link rel="prev" href="lecture-taxonomy-profiling.html"/>
<link rel="next" href="lecture-possible-datasets.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="assets/style.css" type="text/css" />
<link rel="stylesheet" href="assets/C-MOOR_Template/c-moor-ottr.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<a href="https://www.c-moor.org/" target="_blank"><img src="assets/C-MOOR_Template/c-moor-logo-vertical.png" style="width: 70%; padding-left: 10px;"</a>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#learning-goals"><i class="fa fa-check"></i>Learning Goals</a></li>
</ul></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="lecture-welcome-to-your-genomics-adventure.html"><a href="lecture-welcome-to-your-genomics-adventure.html"><i class="fa fa-check"></i><b>1.1</b> Lecture: Welcome to Your Genomics Adventure!</a></li>
<li class="chapter" data-level="1.2" data-path="activity-create-accounts.html"><a href="activity-create-accounts.html"><i class="fa fa-check"></i><b>1.2</b> Activity: Create Accounts</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="activity-create-accounts.html"><a href="activity-create-accounts.html#purpose"><i class="fa fa-check"></i><b>1.2.1</b> Purpose</a></li>
<li class="chapter" data-level="1.2.2" data-path="activity-create-accounts.html"><a href="activity-create-accounts.html#activity"><i class="fa fa-check"></i><b>1.2.2</b> Activity</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="lecture-the-scientic-process.html"><a href="lecture-the-scientic-process.html"><i class="fa fa-check"></i><b>1.3</b> Lecture: The Scientic Process</a></li>
<li class="chapter" data-level="1.4" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html"><i class="fa fa-check"></i><b>1.4</b> Homework: Post Introductions</a>
<ul>
<li class="chapter" data-level="1.4.1" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html#purpose-1"><i class="fa fa-check"></i><b>1.4.1</b> Purpose</a></li>
<li class="chapter" data-level="1.4.2" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html#learning-objectives"><i class="fa fa-check"></i><b>1.4.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="1.4.3" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html#introduction-1"><i class="fa fa-check"></i><b>1.4.3</b> Introduction</a></li>
<li class="chapter" data-level="1.4.4" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html#activity-1---create-a-topic-in-a-private-category"><i class="fa fa-check"></i><b>1.4.4</b> Activity 1 - Create a Topic in a Private Category</a></li>
<li class="chapter" data-level="1.4.5" data-path="homework-post-introductions.html"><a href="homework-post-introductions.html#activity-2---reply-to-a-topic-in-a-public-category"><i class="fa fa-check"></i><b>1.4.5</b> Activity 2 - Reply to a Topic in a Public Category</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Exploration and Discovery</b></span></li>
<li class="chapter" data-level="2" data-path="scientific-literature.html"><a href="scientific-literature.html"><i class="fa fa-check"></i><b>2</b> Scientific Literature</a>
<ul>
<li class="chapter" data-level="2.1" data-path="lecture-whats-in-your-xyz.html"><a href="lecture-whats-in-your-xyz.html"><i class="fa fa-check"></i><b>2.1</b> Lecture: What’s in Your XYZ?</a></li>
<li class="chapter" data-level="2.2" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html"><i class="fa fa-check"></i><b>2.2</b> Activity: Taxonomy Profiling Spreadsheet</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#purpose-2"><i class="fa fa-check"></i><b>2.2.1</b> Purpose</a></li>
<li class="chapter" data-level="2.2.2" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#learning-objectives-1"><i class="fa fa-check"></i><b>2.2.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.2.3" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#introduction-2"><i class="fa fa-check"></i><b>2.2.3</b> Introduction</a></li>
<li class="chapter" data-level="2.2.4" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#activity-1-explore-zymo-gut-standard-metagenomic-diversity"><i class="fa fa-check"></i><b>2.2.4</b> Activity 1 – Explore Zymo Gut Standard Metagenomic Diversity</a></li>
<li class="chapter" data-level="2.2.5" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#activity-2-optional-compare-with-zymo-fecal-reference"><i class="fa fa-check"></i><b>2.2.5</b> Activity 2 (OPTIONAL) – Compare with Zymo Fecal Reference</a></li>
<li class="chapter" data-level="2.2.6" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#grading-criteria-2"><i class="fa fa-check"></i><b>2.2.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="2.2.7" data-path="activity-taxonomy-profiling-spreadsheet.html"><a href="activity-taxonomy-profiling-spreadsheet.html#footnotes-2"><i class="fa fa-check"></i><b>2.2.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="activity-a-short-introduction-to-galaxy.html"><a href="activity-a-short-introduction-to-galaxy.html"><i class="fa fa-check"></i><b>2.3</b> Activity: A short introduction to Galaxy</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="activity-a-short-introduction-to-galaxy.html"><a href="activity-a-short-introduction-to-galaxy.html#introduction-3"><i class="fa fa-check"></i><b>2.3.1</b> Introduction</a></li>
<li class="chapter" data-level="2.3.2" data-path="activity-a-short-introduction-to-galaxy.html"><a href="activity-a-short-introduction-to-galaxy.html#activity-1"><i class="fa fa-check"></i><b>2.3.2</b> Activity</a></li>
<li class="chapter" data-level="2.3.3" data-path="activity-a-short-introduction-to-galaxy.html"><a href="activity-a-short-introduction-to-galaxy.html#grading-criteria-3"><i class="fa fa-check"></i><b>2.3.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="2.3.4" data-path="activity-a-short-introduction-to-galaxy.html"><a href="activity-a-short-introduction-to-galaxy.html#footnotes-3"><i class="fa fa-check"></i><b>2.3.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="homework-scientific-literature-prelab.html"><a href="homework-scientific-literature-prelab.html"><i class="fa fa-check"></i><b>2.4</b> Homework: Scientific Literature Prelab</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="homework-scientific-literature-prelab.html"><a href="homework-scientific-literature-prelab.html#purpose-3"><i class="fa fa-check"></i><b>2.4.1</b> Purpose</a></li>
<li class="chapter" data-level="2.4.2" data-path="homework-scientific-literature-prelab.html"><a href="homework-scientific-literature-prelab.html#learning-objectives-2"><i class="fa fa-check"></i><b>2.4.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.4.3" data-path="homework-scientific-literature-prelab.html"><a href="homework-scientific-literature-prelab.html#introduction-4"><i class="fa fa-check"></i><b>2.4.3</b> Introduction</a></li>
<li class="chapter" data-level="2.4.4" data-path="homework-scientific-literature-prelab.html"><a href="homework-scientific-literature-prelab.html#activity-2"><i class="fa fa-check"></i><b>2.4.4</b> Activity</a></li>
<li class="chapter" data-level="2.4.5" data-path="homework-scientific-literature-prelab.html"><a href="homework-scientific-literature-prelab.html#footnotes-4"><i class="fa fa-check"></i><b>2.4.5</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="discussion-scientific-literature-prelab.html"><a href="discussion-scientific-literature-prelab.html"><i class="fa fa-check"></i><b>2.5</b> Discussion: Scientific Literature Prelab</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="discussion-scientific-literature-prelab.html"><a href="discussion-scientific-literature-prelab.html#activity-3"><i class="fa fa-check"></i><b>2.5.1</b> Activity</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="lecture-scientific-literature.html"><a href="lecture-scientific-literature.html"><i class="fa fa-check"></i><b>2.6</b> Lecture: Scientific Literature</a></li>
<li class="chapter" data-level="2.7" data-path="homework-scientific-literature-activity.html"><a href="homework-scientific-literature-activity.html"><i class="fa fa-check"></i><b>2.7</b> Homework: Scientific Literature Activity</a>
<ul>
<li class="chapter" data-level="2.7.1" data-path="homework-scientific-literature-activity.html"><a href="homework-scientific-literature-activity.html#purpose-4"><i class="fa fa-check"></i><b>2.7.1</b> Purpose</a></li>
<li class="chapter" data-level="2.7.2" data-path="homework-scientific-literature-activity.html"><a href="homework-scientific-literature-activity.html#learning-objectives-3"><i class="fa fa-check"></i><b>2.7.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="2.7.3" data-path="homework-scientific-literature-activity.html"><a href="homework-scientific-literature-activity.html#introduction-5"><i class="fa fa-check"></i><b>2.7.3</b> Introduction</a></li>
<li class="chapter" data-level="2.7.4" data-path="homework-scientific-literature-activity.html"><a href="homework-scientific-literature-activity.html#activity-4"><i class="fa fa-check"></i><b>2.7.4</b> Activity</a></li>
<li class="chapter" data-level="2.7.5" data-path="homework-scientific-literature-activity.html"><a href="homework-scientific-literature-activity.html#results-in-class-and-homework"><i class="fa fa-check"></i><b>2.7.5</b> Results (in class and Homework)</a></li>
<li class="chapter" data-level="2.7.6" data-path="homework-scientific-literature-activity.html"><a href="homework-scientific-literature-activity.html#grading-criteria-5"><i class="fa fa-check"></i><b>2.7.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="2.7.7" data-path="homework-scientific-literature-activity.html"><a href="homework-scientific-literature-activity.html#footnotes-5"><i class="fa fa-check"></i><b>2.7.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="2.8" data-path="presentation-scientific-literature-activity.html"><a href="presentation-scientific-literature-activity.html"><i class="fa fa-check"></i><b>2.8</b> Presentation: Scientific Literature Activity</a>
<ul>
<li class="chapter" data-level="2.8.1" data-path="presentation-scientific-literature-activity.html"><a href="presentation-scientific-literature-activity.html#activity-5"><i class="fa fa-check"></i><b>2.8.1</b> Activity</a></li>
<li class="chapter" data-level="2.8.2" data-path="presentation-scientific-literature-activity.html"><a href="presentation-scientific-literature-activity.html#footnotes-6"><i class="fa fa-check"></i><b>2.8.2</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="3" data-path="microbial-genomes.html"><a href="microbial-genomes.html"><i class="fa fa-check"></i><b>3</b> Microbial Genomes</a>
<ul>
<li class="chapter" data-level="3.1" data-path="lecture-microbial-genome.html"><a href="lecture-microbial-genome.html"><i class="fa fa-check"></i><b>3.1</b> Lecture: Microbial Genome</a></li>
<li class="chapter" data-level="3.2" data-path="homework-microbial-genomes-prelab.html"><a href="homework-microbial-genomes-prelab.html"><i class="fa fa-check"></i><b>3.2</b> Homework: Microbial Genomes Prelab</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="homework-microbial-genomes-prelab.html"><a href="homework-microbial-genomes-prelab.html#purpose-5"><i class="fa fa-check"></i><b>3.2.1</b> Purpose</a></li>
<li class="chapter" data-level="3.2.2" data-path="homework-microbial-genomes-prelab.html"><a href="homework-microbial-genomes-prelab.html#learning-objectives-4"><i class="fa fa-check"></i><b>3.2.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.2.3" data-path="homework-microbial-genomes-prelab.html"><a href="homework-microbial-genomes-prelab.html#activity-1-genbank"><i class="fa fa-check"></i><b>3.2.3</b> Activity 1 – GenBank</a></li>
<li class="chapter" data-level="3.2.4" data-path="homework-microbial-genomes-prelab.html"><a href="homework-microbial-genomes-prelab.html#activity-2-genomes-genes-and-other-databases-bv-brc"><i class="fa fa-check"></i><b>3.2.4</b> Activity 2 – Genomes, genes, and other databases (BV-BRC)</a></li>
<li class="chapter" data-level="3.2.5" data-path="homework-microbial-genomes-prelab.html"><a href="homework-microbial-genomes-prelab.html#activity-3---taxonomy-and-tree-of-life"><i class="fa fa-check"></i><b>3.2.5</b> Activity 3 - Taxonomy and tree of life</a></li>
<li class="chapter" data-level="3.2.6" data-path="homework-microbial-genomes-prelab.html"><a href="homework-microbial-genomes-prelab.html#grading-criteria-6"><i class="fa fa-check"></i><b>3.2.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="3.2.7" data-path="homework-microbial-genomes-prelab.html"><a href="homework-microbial-genomes-prelab.html#footnotes-7"><i class="fa fa-check"></i><b>3.2.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="homework-qc-and-galaxy-workflows.html"><a href="homework-qc-and-galaxy-workflows.html"><i class="fa fa-check"></i><b>3.3</b> Homework: QC and Galaxy Workflows</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="homework-qc-and-galaxy-workflows.html"><a href="homework-qc-and-galaxy-workflows.html#introduction-6"><i class="fa fa-check"></i><b>3.3.1</b> Introduction</a></li>
<li class="chapter" data-level="3.3.2" data-path="homework-qc-and-galaxy-workflows.html"><a href="homework-qc-and-galaxy-workflows.html#activity-1-quality-control-qc"><i class="fa fa-check"></i><b>3.3.2</b> Activity 1 – Quality Control (QC)</a></li>
<li class="chapter" data-level="3.3.3" data-path="homework-qc-and-galaxy-workflows.html"><a href="homework-qc-and-galaxy-workflows.html#activity-2-creating-editing-and-importing-galaxy-workflows"><i class="fa fa-check"></i><b>3.3.3</b> Activity 2 – Creating, Editing and Importing Galaxy Workflows</a></li>
<li class="chapter" data-level="3.3.4" data-path="homework-qc-and-galaxy-workflows.html"><a href="homework-qc-and-galaxy-workflows.html#grading-criteria-7"><i class="fa fa-check"></i><b>3.3.4</b> Grading Criteria</a></li>
<li class="chapter" data-level="3.3.5" data-path="homework-qc-and-galaxy-workflows.html"><a href="homework-qc-and-galaxy-workflows.html#footnotes-8"><i class="fa fa-check"></i><b>3.3.5</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="discussion-microbial-genomes-prelab.html"><a href="discussion-microbial-genomes-prelab.html"><i class="fa fa-check"></i><b>3.4</b> Discussion: Microbial Genomes Prelab</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="discussion-microbial-genomes-prelab.html"><a href="discussion-microbial-genomes-prelab.html#activity-6"><i class="fa fa-check"></i><b>3.4.1</b> Activity</a></li>
<li class="chapter" data-level="3.4.2" data-path="discussion-microbial-genomes-prelab.html"><a href="discussion-microbial-genomes-prelab.html#footnotes-9"><i class="fa fa-check"></i><b>3.4.2</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="homework-microbial-genomes-project.html"><a href="homework-microbial-genomes-project.html"><i class="fa fa-check"></i><b>3.5</b> Homework: Microbial Genomes Project</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="homework-microbial-genomes-project.html"><a href="homework-microbial-genomes-project.html#purpose-6"><i class="fa fa-check"></i><b>3.5.1</b> Purpose</a></li>
<li class="chapter" data-level="3.5.2" data-path="homework-microbial-genomes-project.html"><a href="homework-microbial-genomes-project.html#learning-objectives-5"><i class="fa fa-check"></i><b>3.5.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="3.5.3" data-path="homework-microbial-genomes-project.html"><a href="homework-microbial-genomes-project.html#activity-1-mags-and-taxonomy"><i class="fa fa-check"></i><b>3.5.3</b> Activity 1 – MAGs and Taxonomy</a></li>
<li class="chapter" data-level="3.5.4" data-path="homework-microbial-genomes-project.html"><a href="homework-microbial-genomes-project.html#activity-2-genomes-genes-and-databases"><i class="fa fa-check"></i><b>3.5.4</b> Activity 2 – Genomes, Genes, and Databases</a></li>
<li class="chapter" data-level="3.5.5" data-path="homework-microbial-genomes-project.html"><a href="homework-microbial-genomes-project.html#activity-3-go-deeper"><i class="fa fa-check"></i><b>3.5.5</b> Activity 3 – Go Deeper!</a></li>
<li class="chapter" data-level="3.5.6" data-path="homework-microbial-genomes-project.html"><a href="homework-microbial-genomes-project.html#grading-criteria-8"><i class="fa fa-check"></i><b>3.5.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="3.5.7" data-path="homework-microbial-genomes-project.html"><a href="homework-microbial-genomes-project.html#footnotes-10"><i class="fa fa-check"></i><b>3.5.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="activity-test-driver.html"><a href="activity-test-driver.html"><i class="fa fa-check"></i><b>3.6</b> Activity: test-driveR</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="activity-test-driver.html"><a href="activity-test-driver.html#introduction-8"><i class="fa fa-check"></i><b>3.6.1</b> Introduction</a></li>
<li class="chapter" data-level="3.6.2" data-path="activity-test-driver.html"><a href="activity-test-driver.html#activity-7"><i class="fa fa-check"></i><b>3.6.2</b> Activity</a></li>
<li class="chapter" data-level="3.6.3" data-path="activity-test-driver.html"><a href="activity-test-driver.html#grading-criteria-9"><i class="fa fa-check"></i><b>3.6.3</b> Grading Criteria</a></li>
<li class="chapter" data-level="3.6.4" data-path="activity-test-driver.html"><a href="activity-test-driver.html#footnotes-11"><i class="fa fa-check"></i><b>3.6.4</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="presentation-microbial-genomes-project.html"><a href="presentation-microbial-genomes-project.html"><i class="fa fa-check"></i><b>3.7</b> Presentation: Microbial Genomes Project</a>
<ul>
<li class="chapter" data-level="3.7.1" data-path="presentation-microbial-genomes-project.html"><a href="presentation-microbial-genomes-project.html#activity-8"><i class="fa fa-check"></i><b>3.7.1</b> Activity</a></li>
<li class="chapter" data-level="3.7.2" data-path="presentation-microbial-genomes-project.html"><a href="presentation-microbial-genomes-project.html#footnotes-12"><i class="fa fa-check"></i><b>3.7.2</b> Footnotes</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>Testing Ideas</b></span></li>
<li class="chapter" data-level="4" data-path="taxonomy-profiling.html"><a href="taxonomy-profiling.html"><i class="fa fa-check"></i><b>4</b> Taxonomy Profiling</a>
<ul>
<li class="chapter" data-level="4.1" data-path="lecture-taxonomy-profiling.html"><a href="lecture-taxonomy-profiling.html"><i class="fa fa-check"></i><b>4.1</b> Lecture: Taxonomy Profiling</a></li>
<li class="chapter" data-level="4.2" data-path="homework-taxonomy-profiling-prelab.html"><a href="homework-taxonomy-profiling-prelab.html"><i class="fa fa-check"></i><b>4.2</b> Homework: Taxonomy Profiling Prelab</a>
<ul>
<li class="chapter" data-level="4.2.1" data-path="homework-taxonomy-profiling-prelab.html"><a href="homework-taxonomy-profiling-prelab.html#purpose-7"><i class="fa fa-check"></i><b>4.2.1</b> Purpose</a></li>
<li class="chapter" data-level="4.2.2" data-path="homework-taxonomy-profiling-prelab.html"><a href="homework-taxonomy-profiling-prelab.html#learning-objectives-6"><i class="fa fa-check"></i><b>4.2.2</b> Learning Objectives</a></li>
<li class="chapter" data-level="4.2.3" data-path="homework-taxonomy-profiling-prelab.html"><a href="homework-taxonomy-profiling-prelab.html#introduction-9"><i class="fa fa-check"></i><b>4.2.3</b> Introduction</a></li>
<li class="chapter" data-level="4.2.4" data-path="homework-taxonomy-profiling-prelab.html"><a href="homework-taxonomy-profiling-prelab.html#activity-1-qc-reads"><i class="fa fa-check"></i><b>4.2.4</b> Activity 1 – QC Reads</a></li>
<li class="chapter" data-level="4.2.5" data-path="homework-taxonomy-profiling-prelab.html"><a href="homework-taxonomy-profiling-prelab.html#activity-2-taxonomy-profiling-in-galaxy"><i class="fa fa-check"></i><b>4.2.5</b> Activity 2 – Taxonomy Profiling in Galaxy</a></li>
<li class="chapter" data-level="4.2.6" data-path="homework-taxonomy-profiling-prelab.html"><a href="homework-taxonomy-profiling-prelab.html#grading-criteria-10"><i class="fa fa-check"></i><b>4.2.6</b> Grading Criteria</a></li>
<li class="chapter" data-level="4.2.7" data-path="homework-taxonomy-profiling-prelab.html"><a href="homework-taxonomy-profiling-prelab.html#footnotes-13"><i class="fa fa-check"></i><b>4.2.7</b> Footnotes</a></li>
</ul></li>
<li class="chapter" data-level="4.3" data-path="lecture-possible-datasets.html"><a href="lecture-possible-datasets.html"><i class="fa fa-check"></i><b>4.3</b> Lecture: Possible Datasets</a></li>
<li class="chapter" data-level="4.4" data-path="homework-possible-datasets-activity.html"><a href="homework-possible-datasets-activity.html"><i class="fa fa-check"></i><b>4.4</b> Homework: Possible Datasets Activity</a></li>
<li class="chapter" data-level="4.5" data-path="homework-using-dataset-collections.html"><a href="homework-using-dataset-collections.html"><i class="fa fa-check"></i><b>4.5</b> Homework: Using dataset collections</a></li>
<li class="chapter" data-level="4.6" data-path="discussion-taxonomy-profiling-prelab.html"><a href="discussion-taxonomy-profiling-prelab.html"><i class="fa fa-check"></i><b>4.6</b> Discussion: Taxonomy Profiling Prelab</a></li>
<li class="chapter" data-level="4.7" data-path="homework-taxonomy-profiling-project.html"><a href="homework-taxonomy-profiling-project.html"><i class="fa fa-check"></i><b>4.7</b> Homework: Taxonomy Profiling Project</a></li>
<li class="chapter" data-level="4.8" data-path="activity-meet-r.html"><a href="activity-meet-r.html"><i class="fa fa-check"></i><b>4.8</b> Activity: Meet R!</a></li>
<li class="chapter" data-level="4.9" data-path="presentation-taxonomy-profiling-project.html"><a href="presentation-taxonomy-profiling-project.html"><i class="fa fa-check"></i><b>4.9</b> Presentation: Taxonomy Profiling Project</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="about-the-authors.html"><a href="about-the-authors.html"><i class="fa fa-check"></i>About the Authors</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<p style="text-align:center;"> <a href="https://github.com/jhudsl/OTTR_Template" target="blank" > This content was published with</a> <a href="https://bookdown.org/" target="blank"> bookdown by:</a> </p>
<p style="text-align:center;"> <a href="http://jhudatascience.org/"> The Johns Hopkins Data Science Lab </a></p>
<p style="text-align:center; font-size: 12px;"> <a href="https://github.com/rstudio4edu/rstudio4edu-book/"> Style adapted from: rstudio4edu-book </a> <a href ="https://creativecommons.org/licenses/by/2.0/"> (CC-BY 2.0) </a></p>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Course Name</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<head>
  <meta name="viewport" content="width=device-width,minimum-scale=1.0,maximum-scale=10.0,initial-scale=1.0">
  <!--script src="https://kit.fontawesome.com/6a26f47516.js"></script-->
  <!--<script src="assets/hideOutput.js"></script>-->
</head>
        


<div class="hero-image-container"> 
  <img class= "hero-image" src="assets/C-MOOR_Template/dasl_thin_main_image.png">
</div>
<div id="homework-taxonomy-profiling-prelab" class="section level2 hasAnchor" number="4.2">
<h2><span class="header-section-number">4.2</span> Homework: Taxonomy Profiling Prelab<a href="homework-taxonomy-profiling-prelab.html#homework-taxonomy-profiling-prelab" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="purpose-7" class="section level3 hasAnchor" number="4.2.1">
<h3><span class="header-section-number">4.2.1</span> Purpose<a href="homework-taxonomy-profiling-prelab.html#purpose-7" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>To use a variety of Galaxy tools to perform Quality Control (QC), taxonomy profiling, and visualization of a metagenomics sample.</p>
</div>
<div id="learning-objectives-6" class="section level3 hasAnchor" number="4.2.2">
<h3><span class="header-section-number">4.2.2</span> Learning Objectives<a href="homework-taxonomy-profiling-prelab.html#learning-objectives-6" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Use Galaxy tools to:</p>
<ol style="list-style-type: decimal">
<li>Perform Quality Control (QC) on your raw data by checking the quality of your raw reads</li>
<li>Assign taxonomy labels to your reads</li>
<li>Visualize the classified metagenome</li>
</ol>
<p>In this exercise, using Galaxy, you will be:</p>
<ol style="list-style-type: decimal">
<li>Using NanoPlot tool to examine the quality of your sequencing reads</li>
<li>Running a workflow to perform taxonomy profiling and visualization in a single step</li>
</ol>
</div>
<div id="introduction-9" class="section level3 hasAnchor" number="4.2.3">
<h3><span class="header-section-number">4.2.3</span> Introduction<a href="homework-taxonomy-profiling-prelab.html#introduction-9" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>To find out which microorganisms are present in the sample, it is important to have high-quality DNA sequences. To ensure high-quality sequence input, QC (and in many cases also read trimming and filtering) are routinely performed on raw sequences. The reads can then be used to determine which species, genera, families, and other taxonomic ranks are present in the sample. To assign taxonomy, we can compare the reads of the sample to a reference database, i.e. sequences of known microorganisms stored in a database, using Kraken2, which is a k-mer based taxonomic assignment tool. We can then use a tool like Krona to interactively visualize and explore the composition of a metagenome.</p>
</div>
<div id="activity-1-qc-reads" class="section level3 hasAnchor" number="4.2.4">
<h3><span class="header-section-number">4.2.4</span> Activity 1 – QC Reads<a href="homework-taxonomy-profiling-prelab.html#activity-1-qc-reads" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 50 min</em></p>
<div id="activity-1---part-i-import-dataset-into-galaxy" class="section level4 hasAnchor" number="4.2.4.1">
<h4><span class="header-section-number">4.2.4.1</span> Activity 1 - Part I: Import dataset into Galaxy<a href="homework-taxonomy-profiling-prelab.html#activity-1---part-i-import-dataset-into-galaxy" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><em>Estimated time: 15 min</em></p>
</div>
<div id="instructions-26" class="section level4 hasAnchor" number="4.2.4.2">
<h4><span class="header-section-number">4.2.4.2</span> Instructions<a href="homework-taxonomy-profiling-prelab.html#instructions-26" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Import dataset into Galaxy.</p>
<ol style="list-style-type: lower-alpha">
<li><p>Open the zymo-gut-standard public history <a href="https://usegalaxy.org/u/valerie-g/h/zymo-gut-standard-d6331-subset-1">https://usegalaxy.org/u/valerie-g/h/zymo-gut-standard-d6331-subset-1</a></p></li>
<li><p>Click on <strong>Import this history</strong>, select Copy only the active, non-deleted datasets and then Copy History.</p></li>
<li><p>Confirm <mark style="background color: lightgreen">Zymo_Gut_Standard_D6331_subset</mark> exists in your history by clicking on the Home button “Galaxy” on top left ().</p></li>
<li><p>Click on Zymo_Gut_Standard_D6331_subset to explore content.</p></li>
</ol>
</div>
<div id="questions-14" class="section level4 hasAnchor" number="4.2.4.3">
<h4><span class="header-section-number">4.2.4.3</span> Questions<a href="homework-taxonomy-profiling-prelab.html#questions-14" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<table>
<thead>
<tr class="header">
<th align="left">1. What is the size of this downloaded dataset subset?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><break></p>
<table>
<thead>
<tr class="header">
<th align="left">2. What is the format/extension of the downloaded file?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><break></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3. Click on the Display (eyeball) icon and describe what you see in the 4 lines of the fastq file?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">Line 1:</td>
</tr>
<tr class="even">
<td align="left">Line 2:</td>
</tr>
<tr class="odd">
<td align="left">Line 3:</td>
</tr>
<tr class="even">
<td align="left">Line 4:</td>
</tr>
</tbody>
</table>
<p><break></p>
</div>
<div id="activity-1---part-ii-run-nanoplot-in-galaxy-to-assess-sequence-quality" class="section level4 hasAnchor" number="4.2.4.4">
<h4><span class="header-section-number">4.2.4.4</span> Activity 1 - Part II: Run Nanoplot in Galaxy to assess sequence quality<a href="homework-taxonomy-profiling-prelab.html#activity-1---part-ii-run-nanoplot-in-galaxy-to-assess-sequence-quality" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><em>Estimated time: 15 min</em></p>
</div>
<div id="instructions-27" class="section level4 hasAnchor" number="4.2.4.5">
<h4><span class="header-section-number">4.2.4.5</span> Instructions<a href="homework-taxonomy-profiling-prelab.html#instructions-27" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Run Nanoplot in Galaxy.</p>
<ol style="list-style-type: lower-alpha">
<li><p>Explore NanoPlot tool parameters - click on the Tools icon on the left of the page. Then, in the search bar enter ‘NanoPlot’ and select the <strong>NanoPlot</strong> tool. Explore NanoPlot functionality via examining <strong>Tool Parameters</strong>.</p></li>
<li><p>Run NanoPlot using default settings. Under Tool Parameters, <strong>check the following settings</strong>:</p></li>
</ol>
<ul>
<li><p>Under <strong>files</strong> there are 4 options to select a fastq dataset: Single dataset, Multiple datasets, Dataset collection or option ‘…’ which is Browse or Upload Datasets. Browse to select your fastq dataset. <strong>Note</strong>, Galaxy tool may pre-select the correct dataset already for you so just make sure that the file is correct.</p></li>
<li><p>Click on Run Tool and wait ~5-10 minutes as the NanoPlot job is scheduled, run, and complete.</p></li>
</ul>
</div>
<div id="questions-15" class="section level4 hasAnchor" number="4.2.4.6">
<h4><span class="header-section-number">4.2.4.6</span> Questions<a href="homework-taxonomy-profiling-prelab.html#questions-15" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">1. Under <strong>Type of file(s) to work on</strong>, check to see which input files are compatible with NanoPlot and name 2 file extension options listed.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">File extension name 1:</td>
</tr>
<tr class="even">
<td align="left">File extension name 2:</td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">2. Click to expand <em>Options for filtering or transforming input prior to plotting</em> and name 3 options you could use to filter your sequencing data.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">1.</td>
</tr>
<tr class="even">
<td align="left">2.</td>
</tr>
<tr class="odd">
<td align="left">3.</td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">3. <mark style="background color: yellow"><strong>Run Nanoplot</strong></mark> using default tool settings and record how many output files you obtained after running NanoPlot and list their names.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"># of output files</td>
</tr>
<tr class="even">
<td align="left">Names of output files</td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="activity-1---part-iii-view-nanoplot-results-in-galaxy" class="section level4 hasAnchor" number="4.2.4.7">
<h4><span class="header-section-number">4.2.4.7</span> Activity 1 - Part III: View NanoPlot Results in Galaxy<a href="homework-taxonomy-profiling-prelab.html#activity-1---part-iii-view-nanoplot-results-in-galaxy" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><em>Estimated time: 15 min</em></p>
</div>
<div id="instructions-28" class="section level4 hasAnchor" number="4.2.4.8">
<h4><span class="header-section-number">4.2.4.8</span> Instructions<a href="homework-taxonomy-profiling-prelab.html#instructions-28" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>View and examine NanoPlot Results in Galaxy by clicking on the Display icon (eyeball) next to the NanoPlot output.</p>
</div>
<div id="questions-16" class="section level4 hasAnchor" number="4.2.4.9">
<h4><span class="header-section-number">4.2.4.9</span> Questions<a href="homework-taxonomy-profiling-prelab.html#questions-16" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ol style="list-style-type: decimal">
<li>Click on the Display icon (eyeball) next to the NanoPlot output files to view results.</li>
</ol>
<table>
<thead>
<tr class="header">
<th align="left">A. How many bases were sequenced?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">B. Why is mean read length longer than the median read length? - Hint: think skewness <a href="https://wikipedia.org/wiki/Skewness" class="uri">https://wikipedia.org/wiki/Skewness</a></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">C. Record Reads &gt;Q20 metric value. Given that Q20 quality (Phred) score corresponds to read accuracy of 99% (or 1 in 100 errors), do you think this dataset is of a good sequence quality?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<ol start="2" style="list-style-type: decimal">
<li>Click on the Display icon (eyeball) next to the NanoPlot output HTML report.</li>
</ol>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">A. Scroll down to view the ‘Weighted histogram of read lengths’ histogram. From this plot estimate the range of read lengths obtained</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">B. Scroll down to view the ‘Yield by length’ cumulative plot which shows sequencing yield based on read length. From this plot do shorter (10kb or less) or longer sequences produce more data?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
</div>
<div id="activity-2-taxonomy-profiling-in-galaxy" class="section level3 hasAnchor" number="4.2.5">
<h3><span class="header-section-number">4.2.5</span> Activity 2 – Taxonomy Profiling in Galaxy<a href="homework-taxonomy-profiling-prelab.html#activity-2-taxonomy-profiling-in-galaxy" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><em>Estimated time: 50 min</em></p>
<div id="activity-2---part-i-run-taxonomy-profiling-workflow-in-galaxy" class="section level4 hasAnchor" number="4.2.5.1">
<h4><span class="header-section-number">4.2.5.1</span> Activity 2 - Part I: Run ‘Taxonomy Profiling’ workflow in Galaxy<a href="homework-taxonomy-profiling-prelab.html#activity-2---part-i-run-taxonomy-profiling-workflow-in-galaxy" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><em>Estimated time: 15 min</em></p>
</div>
<div id="instructions-29" class="section level4 hasAnchor" number="4.2.5.2">
<h4><span class="header-section-number">4.2.5.2</span> Instructions<a href="homework-taxonomy-profiling-prelab.html#instructions-29" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ol style="list-style-type: decimal">
<li>Run ‘Taxonomy Profiling’ public workflow</li>
</ol>
<ol style="list-style-type: lower-alpha">
<li><p>Open the taxonomy-profiling public workflow <a href="https://usegalaxy.org/u/cutsort/w/taxonomy-profiling">https://usegalaxy.org/u/cutsort/w/taxonomy-profiling</a> and click on <strong>Run</strong>.</p></li>
<li><p>Browse to select your fastq dataset by clicking on the ‘…’ tab.</p></li>
<li><p>Under <strong>kraken_database</strong> select <mark style="background color: green">’Prebuilt Refseq indexes: PlusPF(Standard plus protozoa and fungi)(Version:2022-06-07 - Downloaded: 2022-09-04T165121Z).</p></li>
<li><p>Click <strong>Run Workflow</strong> with the following parameters:</p></li>
<li><p>Wait ~15-30 minutes as the Kraken2, KrakenTools, and Krona jobs are scheduled, run, and complete.</p></li>
</ol>
<ol start="2" style="list-style-type: decimal">
<li>While the Taxonomy Workflow is running, click on the Tools icon on the left of the page. Then, in the search bar enter ‘Kraken2’’ and select the <strong>Kraken2</strong> tool.</li>
</ol>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">A. How does Galaxy describe the Kraken2 tool in its descriptor on top of the page?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">B. Scroll down to <strong>Help/What it does</strong> section, and in your own words paraphrase the paragraph describing how Kraken 2 works.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">C. Record how many output files you obtained from the Taxonomy Workflow and list their names.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
<div id="activity-2---part-ii-view-kraken2-results" class="section level4 hasAnchor" number="4.2.5.3">
<h4><span class="header-section-number">4.2.5.3</span> Activity 2 - Part II: View Kraken2 results<a href="homework-taxonomy-profiling-prelab.html#activity-2---part-ii-view-kraken2-results" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><em>Estimated time: 15 min</em></p>
</div>
<div id="instructions-30" class="section level4 hasAnchor" number="4.2.5.4">
<h4><span class="header-section-number">4.2.5.4</span> Instructions<a href="homework-taxonomy-profiling-prelab.html#instructions-30" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Click on the Display icon (eyeball) next to the output file with <strong>converted_kraken_report</strong>. This report should look familiar from week 1 taxonomy-profiling-spreadsheet activity.</p>
</div>
<div id="questions-17" class="section level4 hasAnchor" number="4.2.5.5">
<h4><span class="header-section-number">4.2.5.5</span> Questions<a href="homework-taxonomy-profiling-prelab.html#questions-17" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<ol style="list-style-type: decimal">
<li>Without doing a spreadsheet activity to summarize the proportion of unclassified, classified and different taxonomic ranks, just eye/scroll the report to answer the following questions:</li>
</ol>
<table>
<thead>
<tr class="header">
<th align="left">A. How many Unclassified reads are there?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">B. How many Kingdoms are there and what are they?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">C. How many Phyla are there and what are they?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">D. Using the total <em>number_of_reads</em> you obtained from the <strong>NanoStats</strong> (NanoPlot) metrics, and the value of <em>Unclassified</em> reads from the <strong>converted_kraken_report</strong>, calculate % unclassified and % classified taxa.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">E. Instead of the eyeball, now click on the <strong>converted_kraken_report</strong> entry and just below the ‘Add Tags’ it shows the number of lines and columns in the file. Remember that the number of lines corresponds to the number of taxa detected. Excluding Unclassified subset, how many taxa were identified?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<ol start="2" style="list-style-type: decimal">
<li>Examine <strong>kraken2_with_pluspf_database_output_report</strong> by clicking on the Display icon (eyeball) next to the output file.</li>
</ol>
<p>This output report is an extended version of the converted_kraken_report. The output contains 6 columns. See info for select column headers below:</p>
<ul>
<li>Column 1: Percentage (%) of a given taxon</li>
<li>Column 2: # of reads per given taxon</li>
<li>Column 4: A rank code, indicating (U)nclassified, (R)oot, (D)omain, (K)ingdom, (P)hylum, (C)lass, (O)rder, (F)amily, (G)enus, or (S)pecies. Note, that in this extended file, some rank codes will have numbers associated with them; Ignore this aspect of the document for the moment.</li>
<li>Column 6: Identified taxa/scientific name.</li>
</ul>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">A. What is the percentage of Unclassified taxa listed? Does it match what you calculated in section 2-2.1?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">B. What is the percentage of Classified taxa listed? Does it match what you calculated in section 2-2.1?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">C. Find and record the 3 most abundant Phyla (p_) by percentages. As rows are not sorted by abundance, you may find it helpful to search using for “P” using <CTRL>-F to highlight rows corresponding to phyla.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<ol start="3" style="list-style-type: decimal">
<li>View Krona Results</li>
</ol>
<p>Krona pie chart is one of the outputs of the Taxonomy workflow, and it is an interactive visualization tool for exploring the composition of metagenomes.</p>
<table>
<thead>
<tr class="header">
<th align="left">A. What are the 2 main phyla you observe?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<thead>
<tr class="header">
<th align="left">B. What appears to be the more diverse phyla of the two and why?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">C. Examine how many reads and percent of reads classified as phylum Firmicutes.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">D. What is one of the most abundant Orders in phylum FIrmicutes based on number and % of reads?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">E. Within the most abundant Order from your answer above, record one Family member, one genus member and one species member; also include percent abundances.</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
<table>
<colgroup>
<col width="100%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">F. Some potentially pathogenic bacterial species are often present in a healthy gut, but at relatively low amounts. What % of bacteria is represented by <em>C. difficile</em> (<em>Clostridiodes difficile</em>) in this gut standard sample? How does it compare to the expected proportion in <a href="https://files.zymoresearch.com/datasheets/ds1712_zymobiomics_gut_microbiome_standard_data_sheet.pdf?_gl=1*1njkhlx*_gcl_au*MzE1NTQwNzEuMTczODI1OTA5OA..">ZymoBIOMICS® Gut Microbiome Standard</a>?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left"></td>
</tr>
</tbody>
</table>
<p><br></p>
</div>
</div>
<div id="grading-criteria-10" class="section level3 hasAnchor" number="4.2.6">
<h3><span class="header-section-number">4.2.6</span> Grading Criteria<a href="homework-taxonomy-profiling-prelab.html#grading-criteria-10" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Download as Microsoft Word (.docx) and upload on Canvas</p>
</div>
<div id="footnotes-13" class="section level3 hasAnchor" number="4.2.7">
<h3><span class="header-section-number">4.2.7</span> Footnotes<a href="homework-taxonomy-profiling-prelab.html#footnotes-13" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Resources</strong></p>
<ul>
<li>Google Doc</li>
<li>Species composition in the Gut Microbiome Standard dataset: <a href="https://files.zymoresearch.com/datasheets/ds1712_zymobiomics_gut_microbiome_standard_data_sheet.pdf?_gl=1*1njkhlx*_gcl_au*MzE1NTQwNzEuMTczODI1OTA5OA..">ZymoBIOMICS® Gut Microbiome Standard</a></li>
<li>If interested reading more about Kraken 2, see <a href="https://pubmed.gov/31779668">Kraken 2 publication</a></li>
</ul>
<p><strong>Contributions and Affiliations</strong></p>
<ul>
<li>Valeriya Gaysinskaya, Johns Hopkins University</li>
<li>Frederick Tan, Johns Hopkins University</li>
</ul>
<p>Last Revised: February 2025</p>
</div>
</div>
<hr>
<center> 
  <div class="footer">
      All materials <a href= "https://creativecommons.org/licenses/by/4.0/"> CC-BY </a> unless noted otherwise.
  </div>
</center>
            </section>

          </div>
        </div>
      </div>
<a href="lecture-taxonomy-profiling.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="lecture-possible-datasets.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section"
}
});
});
</script>

</body>

</html>
