

# Microbial Genomes

## Lecture: Microbial Genome

![](resources/images/microbial-genomes_files/figure-docx//1bnFhIIu6ZXSCjlrz5qWc7BZNz-7VfYWkKgbz3kL-ZNU_g35f391192_00.png)

[Slides: Microbial Genomes](https://docs.google.com/presentation/d/1bnFhIIu6ZXSCjlrz5qWc7BZNz-7VfYWkKgbz3kL-ZNU/edit?usp=sharing)

## Homework: Microbial Genomes Prelab

### Purpose

Impress all the information that is freely available about well studied (and not so well studied) bacterial species.  Start with E. coli as one of the best and longest studied before going into recently discovered bacteria (with less information).

### Learning Objectives

1. GenBank – Explore sequence database of all publicly available DNA sequences
2. Sequence Browser – Observe genome organization using graphical representation
3. Bacterial database BV-BRC - Explore bacterial genes and their function
4. Taxonomy Browser – Identify relationships between taxa
5. Lifemap - Tree of Live viewer to visualize relationships between taxa

### Activity 1 – GenBank

![](resources/images/microbial-genomes_files/figure-docx//1hYKF7Ss3vJ8rrUIH7ByNh1BUlRa2fhsJhq8MXzEowCc_g305048ba785_0_0.png)

#### Activity 1 - Part I

*Estimated time: 5 min*

#### Instructions

Navigate to **GenBank** [https://www.ncbi.nlm.nih.gov/genbank](https://www.ncbi.nlm.nih.gov/genbank) and enter ‘E. coli strain K-12’ in the search bar (which should default to **Nucleotide**). This will give you a list of almost 300,000 E. coli sequences. Go down the list and click on the entry that corresponds to the complete genome of the substrain MG1655, which should be on page 1! Click on the selection and answer the following questions.

#### Questions

| **1. What is the size of this *E. coli* genome in bp and Kbp**? |
|:-|
| <br> |

| **2. Is the *E. coli* genome linear or circular**? |
|:-|
| <br> |

| **3. What is the ACCESSION number?** |
|:-|
| <br> |

#### Activity 1 - Part II

*Estimated time: 5 min*

#### Instructions

Back in GenBank [https://www.ncbi.nlm.nih.gov/genbank](https://www.ncbi.nlm.nih.gov/genbank), under the **Nucleotide** search tab,  instead of typing ‘E. coli strain K-12’, type in the ACCESSION number you found (NZ_CP169634). 

#### Questions

| **1. How many records do you observe after an ACCESSION number entry?** |
|:-|
| <br> |

| **2. What can you conclude about what is the ACCESSION number?** |
|:-|
| <br> |


### Activity 2 – Genomes, genes, and other databases (BV-BRC)

#### Activity 2 - Part I

*Estimated time: 5 min*

#### Instructions

After entering your accession number, the on the right of the page, under **Related Information**, click on Assembly link to explore genome assembly information.

#### Questions

| **1. How many chromosomes does E. coli have?** |
|:-|
| <br> |


| **2. What is the genome coverage of this sequenced genome?** |
|:-|
| <br> |

| **3. How many genes were annotated for this genome?** |
|:-|
| <br> |

#### Activity 2 - Part II

*Estimated time: 10 min*

#### Instructions

Go back and on the top of the page and click on **Graphics** to explore genome browser.

a. Hover along one of many green vertical sticks.  
- You can also zoom into a smaller and smaller genomic region for higher resolution.

b. Hover along one of many red vertical sticks.
- You can also zoom into a smaller and smaller genomic region for higher resolution.

#### Questions

|**1. What do the ‘green sticks’ represent?**|
|:-|
| <br> |

|**2. What do the ‘red sticks’ represent?**|
|:-|
| <br> |

|**3. Record 5 genes you found present in E. coli**.|
|:-|
| Gene 1
| Gene 2
| Gene 3
| Gene 4
| Gene 5
| <br> |

#### Activity 2 - Part III

*Estimated time: 10 min*

#### Instructions

To learn more about the genes of interest and their function scientists often use specialized databases. One such bacterial database is BV-BRC [https://www.bv-brc.org](https://www.bv-brc.org). Use BV-BRC to find information about some of the E. coli genes. If there are 0 results then indicate <mark style="background-color: lightyellow">No results found.

**For the 3 genes below, and one gene of your choice from activity above**, in the BV-BRC **Search** space, 

**a)** from a dropdown menu select **“Pathways”** , 

**b)** type in the gene name and **click enter**. This will result in a lot of entries for different organisms. 

**c)** To retrieve information specifically for *E. coli (Escherichia coli)*, in the **Keyword** space type “Escherichia”.

**d)** Check one of the boxes corresponding to *E. coli* strains and enter below which E. coli strain **(Genome Name)** you selected. <mark style="background-color: lightyellow">If no E. coli entry is present, select and record another bacterial Genome Name to learn about your gene’s function.

#### Questions

|**1. Record Genome Name associated with the following genes**.| |
|:--|:--| 
|**Gene ID** | **Genome Name** |
| Gene 1: ampC| Escherichia coli 07798 | 
| Gene 2: mgtA| |
| Gene 3: cdd | |
| Gene 4: Your gene| |
| <br> |

|**2. For the 3 genes from the activity above, record gene Product**.| |
|:--|:--| 
|**Gene ID** | **Gene Product** |
| Gene 1: ampC| Beta-lactamase | 
| Gene 2: | |
| Gene 3: | |
| <br> |

|**3. For the 3 genes from the activity above, record Pathway Name (relates to function)**.| |
|:--|:--|
|**Gene ID** | **Pathway Name** |
| Gene 1: ampC| beta-Lactam resistance | 
| Gene 2: | |
| Gene 3: | |
| <br> |

**Note**, For some well characterized genes  you can additionally obtain more detailed information about the gene/protein function. 

- To do so, from the ‘green’ menu on the right, select **FEATURE** option, to learn from the Special Properties section about a special property of your gene. 

### Activity 3 -  Taxonomy and tree of life

*Estimated time: 30 min*

#### Activity 3 - Part I

#### Instructions

a. Back in NCBI enter the *E. coli* accession number again. Under **Related Information** on the right, click on **Taxonomy** and then on the provided link for the *E. coli*. 

b. Find Lineage information. Full Lineage information contains 7 core taxonomy ranks: Kingdom, Phylum, Class, Order, Family, Genus and Species, plus any additional classification ranks. To just get the 7 core lineage names, click on Lineage link for the abbreviated Lineage, or, simply hover over lineage names.

#### Questions

|1. **Record 7 core taxonomy ranks below**.| |
|:--|:--|
| Kingdom: | |
| Phylum:| |
| Class:| |
| Order:| | 
| Family:| |
| Genus:| |
| Species:| |
|<br>|

#### Activity 3 - Part II

#### Instructions

a) As with the BV-BRC database above, we can use another database called **Lifemap** to visually explore the *E. coli* in the context of the tree of life. Go to [https://lifemap-ncbi.univ-lyon1.fr](https://lifemap-ncbi.univ-lyon1.fr), type E. coli and click species tab.

b) On the tree map, the yellow tag will indicate *E. coli*. Use plus and minus tabs to zoom in and out and visualize E. coli relative to other organisms on the map. 

#### Questions

|**1. How many Domains of life are there?**.| 
|:--|
| <br> |

|**2. Zoom into and find nodes for the E. coli Genus, Family, Order, Class and Phylum. Do they match what you found in the NCBI Taxonomy Browser for Lineage?**.| 
|:--|
| <br> |

|**3. What are some other members of the Genus (Escherichia) to which E. coli belongs?**.| 
|:--|
| <br> |

|**4. What are some other members of the Family (Enterobacteriaceae) to which E. coli belongs?**.| 
|:--|
| <br> |

|**5. What are some other members of the Order (Enterobacterales) to which E. coli belongs?**.| 
|:--|
| <br> |

|**6. What are some other members of the Class (Gammaproteobacteria) to which E. coli belongs?**.| 
|:--|
| <br> |

|**7. What are some other members of the Phylum (Pseudomonadota) to which E. coli belongs?**.| 
|:--|
| <br> |

### Grading Criteria

- <mark style="background-color: yellow">Download as Microsoft Word (.docx) and upload on Canvas.

### Footnotes

**Resources**

- Google Doc

**Contributions and Affiliations**

- Valeriya Gaysinskaya, Johns Hopkins University
- Frederick Tan, Johns Hopkins University

Last Revised: February 2025



## Homework: QC and Galaxy Workflows

### Introduction

In the “A short introduction to Galaxy” activity you learned how to upload a file, use a tool, view results, view histories, extract and run a workflow, and share a history.  You will now practice using these skills to do one of the first things when encountering a new sequencing dataset – quality control (QC).  After assessing the quality of both short and long reads, gain more practice with workflows by creating and editing a new workflow.

### Activity 1 – Quality Control (QC)

*Estimated time: 90 min*

#### Instructions

1. Complete the “Quality Control” hands-on tutorial: [training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/tutorial](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/tutorial.html)

- **NOTE**: Do all hands-on steps in a single Galaxy history

### Activity 2 – Creating, Editing and Importing Galaxy Workflows

*Estimated time: 30 min*

#### Instructions

1. Complete the “Creating, Editing and Importing Galaxy Workflows” hands-on tutorial: [training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-editor/tutorial](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-editor/tutorial.html)

### Grading Criteria

- <mark style="background-color: yellow">Submit URL to your shared Galaxy “Quality Control” history on Canvas

### Footnotes

**Resources**

- Using Galaxy and Managing your Data [topic](https://training.galaxyproject.org/training-material/topics/galaxy-interface/)
- Introduction to Galaxy and Sequence analysis [pathway](https://training.galaxyproject.org/training-material/learning-pathways/intro-to-galaxy-and-genomics.html)

**Contributions and Affiliations**

- Frederick Tan, Johns Hopkins University

Last Revised: February 2025


## Discussion: Microbial Genomes Prelab


### Activity 

*Estimated time: 25 min*

#### Instructions

1. Form new groups of four [here](https://docs.google.com/spreadsheets/d/11eoJgm9mehxGWWzh8IZYDCDmnCmSyshopPYHewvpC8c/edit?usp=sharing).

2. Pair up into groups (10 min).

a. Discuss -- Each group member briefly describes answers to prelab assignment.

b. Summarize -- Identify best answer and add to slidedeck [here](https://docs.google.com/presentation/d/1PxqLt-MDFCTSBWm0-j-mY268b3YUiPMRqYMtcJ5JDL4/edit?usp=sharing)

3. Share group discussion (2 min each group)

### Footnotes

**Contributions and Affiliations**

- Valeriya Gaysinskaya, Johns Hopkins University
- Frederick Tan, Johns Hopkins University

Last Revised: February 2025


## Homework: Microbial Genomes Project

### Purpose

Explore information about new bacterial MAGs from Zue Z et al, Nature Scientific Data 2023 [https://pubmed.ncbi.nlm.nih.gov/37563185/](https://pubmed.ncbi.nlm.nih.gov/37563185/).

### Learning Objectives

1. Explore and better understand MAGs and contigs by following up one MAG and one contig in NCBI.
2. Utilize knowledge learned by navigating MAG information using previously learned tools such as GenBank, Sequence Browser, BV-BRC, Taxonomy Browser and Lifemap. 
3. Go deeper and see what you can find on the organism from your MAG of choice and a gene from a MAG of choice using any tools available to you (Pubmed, Google, BV-BRC, other databases). 

### Activity 1 – MAGs and Taxonomy

*Estimated time: [40] min*

#### Activity 1 - Part I: Explore a MAG

#### Instructions 

- In the research study by Zue Z et al, Nature Scientific Data 2023 [https://pubmed.ncbi.nlm.nih.gov/37563185/](https://pubmed.ncbi.nlm.nih.gov/37563185/), 17 of 103 uncovered MAGs from ballast water or sediment were of very high quality and completeness. You will follow up on one of them!
- **Choose one of these 17 MAGs** (see 17 GenBank IDs below) to follow up in this activity.
- Use GenBank [https://www.ncbi.nlm.nih.gov/nucleotide/](https://www.ncbi.nlm.nih.gov/nucleotide/) to answer questions. 

|GenBank IDs for 17 quality MAGs| 
|:-|
| 1. GCF_030147545.1
| 2. GCF_030148435.1
| 3. GCF_030149225.1
| 4. GCF_030148195.1
| 5. GCF_030148855.1
| 6. GCF_030148245.1
| 7. GCF_030149045.1
| 8. GCF_030148385.1
| 9. GCF_030147715.1
| 10. GCF_030149085.1
| 11. GCF_030149145.1
| 12. GCF_030148125.1
| 13. GCF_030149425.1
| 14. GCF_030149235.1
| 15. GCF_030148515.1
| 16. GCF_030147875.1
| 17. GCF_030149465.1
<br>

#### Questions

|**1. Record the GenBank ID number for one of  17 high quality MAGs for follow up below**.| 
|:--|
| <br> |

<>Here is a snapshot for the 1st MAG (GCF_030147545.1)

![](resources/images/microbial-genomes_files/figure-docx//1hYKF7Ss3vJ8rrUIH7ByNh1BUlRa2fhsJhq8MXzEowCc_g305048ba785_0_77.png)

In the GenBank [https://www.ncbi.nlm.nih.gov/nucleotide/](https://www.ncbi.nlm.nih.gov/nucleotide/), under the Nucleotide search tab **enter** the GenBank number of your MAG and click **Search**.

|**2. Record below the GENOME name for the MAG assembly associated with the GenBank ID you entered.**| 
|:--|
| <br> | 

<>Here is an example for the 1st MAG (GCF_030147545.1), the GENOME name is "Alcanivorax sp. genome ASM3014754v1":

![](resources/images/microbial-genomes_files/figure-docx//1hYKF7Ss3vJ8rrUIH7ByNh1BUlRa2fhsJhq8MXzEowCc_g357b6795023_0_2.png)

Click on the GENOME name of your MAG to **explore genome assembly summary** information.

|**3. What is the Taxon of your MAG?**| 
|:--|
| <br> | 

|**4. What is the genome size of your MAG?**| 
|:--|
| <br> | 

|**5. How many contigs contributed to your MAG assembly?**| 
|:--|
| <br> | 

|**6. How many genes were annotated?**| 
|:--|
| <br> | 

|**7. How comparable is your MAG genome to the E. coli genome from the microbial-genomes-pre-lab, in terms of genome size and number of genes?**| 
|:--|
| <br> | 

|**8 As you can see, MAGs are made up of Contigs. Based on your lecture and reading material, and this exercise, in your own words define MAGs and Contigs below.**| 
|:--|
| MAGs:
| Contigs:
|<br>|


#### Activity 1 - Part II: Explore a Contig

#### Instructions

- MAGs are made up of Contigs. To see which contigs make up your MAG, go back to GenBank [https://www.ncbi.nlm.nih.gov/nucleotide/](https://www.ncbi.nlm.nih.gov/nucleotide/), and search for your MAG using its GenBank ID.

<>Here is an example for the 1st MAG (GCF_030147545.1) composed of 42 contigs:

![](resources/images/microbial-genomes_files/figure-docx//1hYKF7Ss3vJ8rrUIH7ByNh1BUlRa2fhsJhq8MXzEowCc_g357f32d72ad_0_0.png)

|**1. Each contig has its own accession number. Choose a contig of reasonably large size (> 75 kb), click on the contig, then find and record below the Contig’s ID (Accession number/GenBank ID) for further examination.|
|:--|
|Tip: You can sort the entries by length via’ Sort by Sequence Length’ on top!|
|<br>|

- For the contig you chose in activity above, under **Related Information** on the right, click on **Taxonomy** and **then click again on the provided link**.

- Find Lineage information. Full Lineage information contains 7 core taxonomy ranks: Kingdom, Phylum, Class, Order, Family, Genus and Species, plus any additional classification ranks. To just get the 7 core lineage names, click on Lineage link for the abbreviated Lineage, or, simply hover over lineage names. 

|**2. Record 7 core taxonomy ranks for your Contig.|
|:--|:--|
| Kingdom: | |
| Phylum:| |
| Class:| |
| Order:| | 
| Family:| |
| Genus:| |
| Species:| |
|<br>|


#### Activity 1 - Part III: Visualize contig in a tree of life

#### Introductions

a) Use Lifemap to visually explore the contig taxonomy in the context of the tree of life. Go to [https://lifemap-ncbi.univ-lyon1.fr](https://lifemap-ncbi.univ-lyon1.fr) and **enter the lowest taxonomy rank observed for your contig** (most likely the species or genus level, but can also correspond to order or family).

b) On the tree map, use plus and minus tabs to zoom in and out and visualize your Contig entry relative to other organisms on the map. Zoom in and find nodes corresponding to  the higher taxonomic ranks. For example, if your contig corresponds to genus level classification, you will not be able to identify species level information, but you will be able to identify the corresponding Family, Order, Class and Phylum. 

|**1. Record 7 core taxonomy ranks for your Contig.|
|:--|:--|
| Kingdom: | |
| Phylum:| |
| Class:| |
| Order:| | 
| Family:| |
| Genus:| |
| Species:| |
|<br>|

|**2. What are some other members of the Genus to which your Contig belongs?**|
|:--|
|<br>|

|**3. What are some other members of the Family to which your Contig belongs?**|
|:--|
|<br>|

|**4. What are some other members of the Order to which your Contig belongs?**|
|:--|
|<br>|

|**5. What are some other members of the Class to which your Contig belongs?**|
|:--|
|<br>|

|**6. What are some other members of the Phylum to which your Contig belongs?**|
|:--|
|<br>|


### Activity 2 - – Genomes, Genes, and Databases

*Estimated time: [20] min*

#### Instructions

For the contig you chose in activity 1 click on **Graphics** to explore the genome browser and the genes. Record 5 genes of interest. 


## Activity: test-driveR


## Presentation: Microbial Genomes Project
