

# Microbial Genomes

## Lecture: Microbial Genome

![](resources/images/microbial-genomes_files/figure-docx//1bnFhIIu6ZXSCjlrz5qWc7BZNz-7VfYWkKgbz3kL-ZNU_g35f391192_00.png)

[Slides: Microbial Genomes](https://docs.google.com/presentation/d/1bnFhIIu6ZXSCjlrz5qWc7BZNz-7VfYWkKgbz3kL-ZNU/edit?usp=sharing)

## Homework: Microbial Genomes Prelab

### Purpose

Impress all the information that is freely available about well studied (and not so well studied) bacterial species.  Start with E. coli as one of the best and longest studied before going into recently discovered bacteria (with less information).

### Learning Objectives

1. GenBank – Explore sequence database of all publicly available DNA sequences
2. Sequence Browser – Observe genome organization using graphical representation
3. Bacterial database BV-BRC - Explore bacterial genes and their function
4. Taxonomy Browser – Identify relationships between taxa
5. Lifemap - Tree of Live viewer to visualize relationships between taxa

### Activity 1 – GenBank

*Estimated time: 15 min*

### Activity 1 - Part I

#### Instructions

Navigate to **GenBank** [https://www.ncbi.nlm.nih.gov/genbank](https://www.ncbi.nlm.nih.gov/genbank) and enter ‘E. coli strain K-12’ in the search bar (which should default to **Nucleotide**). This will give you a list of almost 300,000 E. coli sequences. Go down the list and click on the entry that corresponds to the complete genome of the substrain MG1655, which should be on page 1! Click on the selection and answer the following questions.

#### Questions

| **1. What is the size of this *E. coli* genome in bp and Kbp**? |
|:-|
| <br> |

| **2. Is the *E. coli* genome linear or circular**? |
|:-|
| <br> |

| **3. What is the ACCESSION number?** |
|:-|
| <br> |

### Activity 1 - Part II

#### Instructions

Back in GenBank [https://www.ncbi.nlm.nih.gov/genbank](https://www.ncbi.nlm.nih.gov/genbank), under the **Nucleotide** search tab,  instead of typing ‘E. coli strain K-12’, type in the ACCESSION number you found (NZ_CP169634). 

#### Questions

| **1. How many records do you observe after an ACCESSION number entry?** |
|:-|
| <br> |

| **2. What can you conclude about what is the ACCESSION number?** |
|:-|
| <br> |

### Activity 2 – Genomes, genes, and other databases (BV-BRC)

*Estimated time: 45 min*

### Activity 2 - Part I

#### Instructions

After entering your accession number, the on the right of the page, under **Related Information**, click on Assembly link to explore genome assembly information.

| **1. How many chromosomes does E. coli have?** |
|:-|
| <br> |


| **2. What is the genome coverage of this sequenced genome?** |
|:-|
| <br> |

| **3. How many genes were annotated for this genome?** |
|:-|
| <br> |

### Activity 2 - Part II

#### Instructions

Go back and on the top of the page and click on **Graphics** to explore genome browser.

a. Hover along one of many green vertical sticks.  
- You can also zoom into a smaller and smaller genomic region for higher resolution.

b. Hover along one of many red vertical sticks.
- You can also zoom into a smaller and smaller genomic region for higher resolution.

|**1. What do the ‘green sticks’ represent?**
|:-|
| <br> |

|**2. What do the ‘red sticks’ represent?**
|:-|
| <br> |

|**3. Record 5 genes you found present in E. coli**.|
|:-|
| Gene 1
| Gene 2
| Gene 3
| Gene 4
| Gene 5
| <br> |

### Activity 2 - Part III

#### Instructions

To learn more about the genes of interest and their function scientists often use specialized databases. One such bacterial database is BV-BRC [https://www.bv-brc.org](https://www.bv-brc.org). Use BV-BRC to find information about some of the E. coli genes. If there are 0 results then indicate ==No results found==.

**For the 3 genes below, and one gene of your choice from activity above**, in the BV-BRC **Search** space, 

**a)** from a dropdown menu select **“Pathways”** , 

**b)** type in the gene name and **click enter**. This will result in a lot of entries for different organisms. 

**c)** To retrieve information specifically for *E. coli (Escherichia coli)*, in the **Keyword** space type “Escherichia”.

**d)** Check one of the boxes corresponding to *E. coli* strains and enter below which E. coli strain **(Genome Name)** you selected.  ==If no E. coli entry is present, select and record another bacterial Genome Name to learn about your gene’s function.==

#### Questions

|**1. Record Genome Name associated with the following genes**.| |
|Gene ID | Genome Name
| Gene 1: ampC| Escherichia coli 07798 | 
| Gene 2: mgtA| |
| Gene 3: cdd | |
| Gene 4: Your gene| |
| <br> |

|**2. For the 3 genes from the activity above, record gene Product.**.| |
|Gene ID | Gene Product
| Gene 1: ampC| Beta-lactamase | 
| Gene 2: | |
| Gene 3: | |
| <br> |

|**3. For the 3 genes from the activity above, record Pathway Name, which relates to gene/protein function.**.| |
|Gene ID | Pathway Name
| Gene 1: ampC| beta-Lactam resistance | 
| Gene 2: | |
| Gene 3: | |
| <br> |

**Note**, For some well characterized genes  you can additionally obtain more detailed information about the gene/protein function. 
- To do so, from the ‘green’ menu on the right, select **FEATURE** option, to learn from the Special Properties section about a special property of your gene. 

## Homework: QC and Galaxy Workflows

### Introduction

In the “A short introduction to Galaxy” activity you learned how to upload a file, use a tool, view results, view histories, extract and run a workflow, and share a history.  You will now practice using these skills to do one of the first things when encountering a new sequencing dataset – quality control (QC).  After assessing the quality of both short and long reads, gain more practice with workflows by creating and editing a new workflow.

### Activity 1 – Quality Control (QC)

*Estimated time: 90 min*

#### Instructions

1. Complete the “Quality Control” hands-on tutorial: [training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/tutorial](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/tutorial.html)

- **NOTE**: Do all hands-on steps in a single Galaxy history

### Activity 2 – Creating, Editing and Importing Galaxy Workflows

*Estimated time: 30 min*

#### Instructions

1. Complete the “Creating, Editing and Importing Galaxy Workflows” hands-on tutorial: [training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-editor/tutorial](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-editor/tutorial.html)

### Grading Criteria

- Submit URL to your shared Galaxy “Quality Control” history on Canvas

### Footnotes

**Resources**

- Using Galaxy and Managing your Data [topic](https://training.galaxyproject.org/training-material/topics/galaxy-interface/)
- Introduction to Galaxy and Sequence analysis [pathway](https://training.galaxyproject.org/training-material/learning-pathways/intro-to-galaxy-and-genomics.html)

**Contributions and Affiliations**

- Frederick Tan, Johns Hopkins University

Last Revised: February 2025




## Discussion: Microbial Genomes Prelab


## Homework: Microbial Genomes Project


## Activity: test-driveR


## Presentation: Microbial Genomes Project
